Binary files tmp/pciutils-3.1.2/common.o and work/main.d/pciutils-3.1.2/common.o differ
Binary files tmp/pciutils-3.1.2/example and work/main.d/pciutils-3.1.2/example differ
Binary files tmp/pciutils-3.1.2/example.o and work/main.d/pciutils-3.1.2/example.o differ
Binary files tmp/pciutils-3.1.2/lib/access.o and work/main.d/pciutils-3.1.2/lib/access.o differ
Binary files tmp/pciutils-3.1.2/lib/caps.o and work/main.d/pciutils-3.1.2/lib/caps.o differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/lib/config.h work/main.d/pciutils-3.1.2/lib/config.h
--- tmp/pciutils-3.1.2/lib/config.h	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/lib/config.h	2009-04-29 10:31:33.000000000 -0300
@@ -0,0 +1,15 @@
+#define PCI_CONFIG_H
+#define PCI_ARCH_I386
+#define PCI_OS_LINUX
+#define PCI_HAVE_PM_LINUX_SYSFS
+#define PCI_HAVE_PM_LINUX_PROC
+#define PCI_HAVE_LINUX_BYTEORDER_H
+#define PCI_PATH_PROC_BUS_PCI "/proc/bus/pci"
+#define PCI_PATH_SYS_BUS_PCI "/sys/bus/pci"
+#define PCI_HAVE_PM_INTEL_CONF
+#define PCI_HAVE_64BIT_ADDRESS
+#define PCI_HAVE_PM_DUMP
+#define PCI_IDS "pci.ids"
+#define PCI_PATH_IDS_DIR "/usr/share"
+#define PCI_SHARED_LIB
+#define PCILIB_VERSION "3.1.2"
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/lib/config.mk work/main.d/pciutils-3.1.2/lib/config.mk
--- tmp/pciutils-3.1.2/lib/config.mk	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/lib/config.mk	2009-04-29 10:31:33.000000000 -0300
@@ -0,0 +1,21 @@
+WITH_LIBS=
+PCILIB=$(LIBNAME).so.$(VERSION)
+LDLIBS=
+LIB_LDLIBS=$(WITH_LIBS)
+SONAME=-Wl,-soname,$(LIBNAME).so$(ABI_VERSION)
+PCILIBPC=$(LIBNAME).pc
+PCI_CONFIG_H=1
+PCI_ARCH_I386=1
+PCI_OS_LINUX=1
+PCI_HAVE_PM_LINUX_SYSFS=1
+PCI_HAVE_PM_LINUX_PROC=1
+PCI_HAVE_LINUX_BYTEORDER_H=1
+PCI_PATH_PROC_BUS_PCI=/proc/bus/pci
+PCI_PATH_SYS_BUS_PCI=/sys/bus/pci
+PCI_HAVE_PM_INTEL_CONF=1
+PCI_HAVE_64BIT_ADDRESS=1
+PCI_HAVE_PM_DUMP=1
+PCI_IDS=pci.ids
+PCI_PATH_IDS_DIR=/usr/share
+PCI_SHARED_LIB=1
+PCILIB_VERSION=3.1.2
Binary files tmp/pciutils-3.1.2/lib/dump.o and work/main.d/pciutils-3.1.2/lib/dump.o differ
Binary files tmp/pciutils-3.1.2/lib/filter.o and work/main.d/pciutils-3.1.2/lib/filter.o differ
Binary files tmp/pciutils-3.1.2/lib/generic.o and work/main.d/pciutils-3.1.2/lib/generic.o differ
Binary files tmp/pciutils-3.1.2/lib/i386-ports.o and work/main.d/pciutils-3.1.2/lib/i386-ports.o differ
Binary files tmp/pciutils-3.1.2/lib/init.o and work/main.d/pciutils-3.1.2/lib/init.o differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/lib/libpci.pc work/main.d/pciutils-3.1.2/lib/libpci.pc
--- tmp/pciutils-3.1.2/lib/libpci.pc	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/lib/libpci.pc	2009-04-29 10:31:39.000000000 -0300
@@ -0,0 +1,10 @@
+prefix=/usr
+includedir=/usr/include
+libdir=/usr/lib
+idsdir=/usr/share
+
+Name: libpci
+Description: libpci
+Version: 3.1.2
+Libs: -L${libdir} -lpci 
+Cflags: -I${includedir}
Binary files tmp/pciutils-3.1.2/lib/libpci.so.3.1.2 and work/main.d/pciutils-3.1.2/lib/libpci.so.3.1.2 differ
Binary files tmp/pciutils-3.1.2/lib/names-cache.o and work/main.d/pciutils-3.1.2/lib/names-cache.o differ
Binary files tmp/pciutils-3.1.2/lib/names-hash.o and work/main.d/pciutils-3.1.2/lib/names-hash.o differ
Binary files tmp/pciutils-3.1.2/lib/names-net.o and work/main.d/pciutils-3.1.2/lib/names-net.o differ
Binary files tmp/pciutils-3.1.2/lib/names.o and work/main.d/pciutils-3.1.2/lib/names.o differ
Binary files tmp/pciutils-3.1.2/lib/names-parse.o and work/main.d/pciutils-3.1.2/lib/names-parse.o differ
Binary files tmp/pciutils-3.1.2/lib/params.o and work/main.d/pciutils-3.1.2/lib/params.o differ
Binary files tmp/pciutils-3.1.2/lib/proc.o and work/main.d/pciutils-3.1.2/lib/proc.o differ
Binary files tmp/pciutils-3.1.2/lib/sysfs.o and work/main.d/pciutils-3.1.2/lib/sysfs.o differ
Binary files tmp/pciutils-3.1.2/ls-caps.o and work/main.d/pciutils-3.1.2/ls-caps.o differ
Binary files tmp/pciutils-3.1.2/ls-ecaps.o and work/main.d/pciutils-3.1.2/ls-ecaps.o differ
Binary files tmp/pciutils-3.1.2/ls-kernel.o and work/main.d/pciutils-3.1.2/ls-kernel.o differ
Binary files tmp/pciutils-3.1.2/ls-map.o and work/main.d/pciutils-3.1.2/ls-map.o differ
Binary files tmp/pciutils-3.1.2/lspci and work/main.d/pciutils-3.1.2/lspci differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/lspci.8 work/main.d/pciutils-3.1.2/lspci.8
--- tmp/pciutils-3.1.2/lspci.8	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/lspci.8	2009-04-29 10:31:40.000000000 -0300
@@ -0,0 +1,354 @@
+.TH lspci 8 "01 February 2009" "pciutils-3.1.2" "The PCI Utilities"
+.IX lspci
+.SH NAME
+lspci \- list all PCI devices
+.SH SYNOPSIS
+.B lspci
+.RB [ options ]
+.SH DESCRIPTION
+.B lspci
+is a utility for displaying information about PCI buses in the system and
+devices connected to them.
+
+By default, it shows a brief list of devices. Use the options described
+below to request either a more verbose output or output intended for
+parsing by other programs.
+
+If you are going to report bugs in PCI device drivers or in
+.I lspci
+itself, please include output of "lspci -vvx" or even better "lspci -vvxxx"
+(however, see below for possible caveats).
+
+Some parts of the output, especially in the highly verbose modes, are probably
+intelligible only to experienced PCI hackers. For exact definitions of
+the fields, please consult either the PCI specifications or the
+.B header.h
+and
+.B /usr/include/linux/pci.h
+include files.
+
+Access to some parts of the PCI configuration space is restricted to root
+on many operating systems, so the features of
+.I lspci
+available to normal users are limited. However,
+.I lspci
+tries its best to display as much as available and mark all other
+information with
+.I <access denied>
+text.
+
+.SH OPTIONS
+
+.SS Basic display modes
+.TP
+.B -m
+Dump PCI device data in a backward-compatible machine readable form.
+See below for details.
+.TP
+.B -mm
+Dump PCI device data in a machine readable form for easy parsing by scripts.
+See below for details.
+.TP
+.B -t
+Show a tree-like diagram containing all buses, bridges, devices and connections
+between them.
+
+.SS Display options
+.TP
+.B -v
+Be verbose and display detailed information about all devices.
+.TP
+.B -vv
+Be very verbose and display more details. This level includes everything deemed
+useful.
+.TP
+.B -vvv
+Be even more verbose and display everything we are able to parse,
+even if it doesn't look interesting at all (e.g., undefined memory regions).
+.TP
+.B -k
+Show kernel drivers handling each device and also kernel modules capable of handling it.
+Turned on by default when
+.B -v
+is given in the normal mode of output.
+(Currently works only on Linux with kernel 2.6 or newer.)
+.TP
+.B -x
+Show hexadecimal dump of the standard part of the configuration space (the first
+64 bytes or 128 bytes for CardBus bridges).
+.TP
+.B -xxx
+Show hexadecimal dump of the whole PCI configuration space. It is available only to root
+as several PCI devices
+.B crash
+when you try to read some parts of the config space (this behavior probably
+doesn't violate the PCI standard, but it's at least very stupid). However, such
+devices are rare, so you needn't worry much.
+.TP
+.B -xxxx
+Show hexadecimal dump of the extended (4096-byte) PCI configuration space available
+on PCI-X 2.0 and PCI Express buses.
+.TP
+.B -b
+Bus-centric view. Show all IRQ numbers and addresses as seen by the cards on the
+PCI bus instead of as seen by the kernel.
+.TP
+.B -D
+Always show PCI domain numbers. By default, lspci suppresses them on machines which
+have only domain 0.
+
+.SS Options to control resolving ID's to names
+.TP
+.B -n
+Show PCI vendor and device codes as numbers instead of looking them up in the
+PCI ID list.
+.TP
+.B -nn
+Show PCI vendor and device codes as both numbers and names.
+.TP
+.B -q
+Use DNS to query the central PCI ID database if a device is not found in the local
+.B pci.ids
+file. If the DNS query succeeds, the result is cached in
+.B ~/.pciids-cache
+and it is recognized in subsequent runs even if
+.B -q
+is not given any more. Please use this switch inside automated scripts only
+with caution to avoid overloading the database servers.
+.TP
+.B -qq
+Same as
+.BR -q ,
+but the local cache is reset.
+.TP
+.B -Q
+Query the central database even for entries which are recognized locally.
+Use this if you suspect that the displayed entry is wrong.
+
+.SS Options for selection of devices
+.TP
+.B -s [[[[<domain>]:]<bus>]:][<slot>][.[<func>]]
+Show only devices in the specified domain (in case your machine has several host bridges,
+they can either share a common bus number space or each of them can address a PCI domain
+of its own; domains are numbered from 0 to ffff), bus (0 to ff), slot (0 to 1f) and function (0 to 7).
+Each component of the device address can be omitted or set to "*", both meaning "any value". All numbers are
+hexadecimal.  E.g., "0:" means all devices on bus 0, "0" means all functions of device 0
+on any bus, "0.3" selects third function of device 0 on all buses and ".4" shows only
+the fourth function of each device.
+.TP
+.B -d [<vendor>]:[<device>]
+Show only devices with specified vendor and device ID. Both ID's are given in
+hexadecimal and may be omitted or given as "*", both meaning "any value".
+
+.SS Other options
+.TP
+.B -i <file>
+Use
+.B
+<file>
+as the PCI ID list instead of /usr/share/pci.ids.
+.TP
+.B -p <file>
+Use
+.B
+<file>
+as the map of PCI ID's handled by kernel modules. By default, lspci uses
+.RI /lib/modules/ kernel_version /modules.pcimap.
+Applies only to Linux systems with recent enough module tools.
+.TP
+.B -M
+Invoke bus mapping mode which performs a thorough scan of all PCI devices, including
+those behind misconfigured bridges etc. This option is available only to root and it
+gives meaningful results only if combined with direct hardware access mode (otherwise
+the results are identical to normal listing modes, modulo bugs in lspci). Please note
+that the bus mapper doesn't support PCI domains and scans only domain 0.
+.TP
+.B --version
+Shows
+.I lspci
+version. This option should be used stand-alone.
+
+.SS PCI access options
+.PP
+The PCI utilities use the PCI library to talk to PCI devices (see
+\fBpcilib\fP(7) for details). You can use the following options to
+influence its behavior:
+.TP
+.B -A <method>
+The library supports a variety of methods to access the PCI hardware.
+By default, it uses the first access method available, but you can use
+this option to override this decision. See \fB-A help\fP for a list of
+available methods and their descriptions.
+.TP
+.B -O <param>=<value>
+The behavior of the library is controlled by several named parameters.
+This option allows to set the value of any of the parameters. Use \fB-O help\fP
+for a list of known parameters and their default values.
+.TP
+.B -H1
+Use direct hardware access via Intel configuration mechanism 1.
+(This is a shorthand for \fB-A intel-conf1\fP.)
+.TP
+.B -H2
+Use direct hardware access via Intel configuration mechanism 2.
+(This is a shorthand for \fB-A intel-conf2\fP.)
+.TP
+.B -F <file>
+Instead of accessing real hardware, read the list of devices and values of their
+configuration registers from the given file produced by an earlier run of lspci -x.
+This is very useful for analysis of user-supplied bug reports, because you can display
+the hardware configuration in any way you want without disturbing the user with
+requests for more dumps.
+.TP
+.B -G
+Increase debug level of the library.
+
+.SH MACHINE READABLE OUTPUT
+If you intend to process the output of lspci automatically, please use one of the
+machine-readable output formats
+.RB ( -m ,
+.BR -vm ,
+.BR -vmm )
+described in this section. All other formats are likely to change
+between versions of lspci.
+
+.P
+All numbers are always printed in hexadecimal. If you want to process numeric ID's instead of
+names, please add the
+.B -n
+switch.
+
+.SS Simple format (-m)
+
+In the simple format, each device is described on a single line, which is
+formatted as parameters suitable for passing to a shell script, i.e., values
+separated by whitespaces, quoted and escaped if necessary.
+Some of the arguments are positional: slot, class, vendor name, device name,
+subsystem vendor name and subsystem name (the last two are empty if
+the device has no subsystem); the remaining arguments are option-like:
+
+.TP
+.BI -r rev
+Revision number.
+
+.TP
+.BI -p progif
+Programming interface.
+
+.P
+The relative order of positional arguments and options is undefined.
+New options can be added in future versions, but they will always
+have a single argument not separated from the option by any spaces,
+so they can be easily ignored if not recognized.
+
+.SS Verbose format (-vmm)
+
+The verbose output is a sequence of records separated by blank lines.
+Each record describes a single device by a sequence of lines, each line
+containing a single
+.RI ` tag :
+.IR value '
+pair. The
+.I tag
+and the
+.I value
+are separated by a single tab character.
+Neither the records nor the lines within a record are in any particular order.
+Tags are case-sensitive.
+
+.P
+The following tags are defined:
+
+.TP
+.B Slot
+The name of the slot where the device resides
+.RI ([ domain :] bus : device . function ).
+This tag is always the first in a record.
+
+.TP
+.B Class
+Name of the class.
+
+.TP
+.B Vendor
+Name of the vendor.
+
+.TP
+.B Device
+Name of the device.
+
+.TP
+.B SVendor
+Name of the subsystem vendor (optional).
+
+.TP
+.B SDevice
+Name of the subsystem (optional).
+
+.TP
+.B PhySlot
+The physical slot where the device resides (optional, Linux only).
+
+.TP
+.B Rev
+Revision number (optional).
+
+.TP
+.B ProgIf
+Programming interface (optional).
+
+.TP
+.B Driver
+Kernel driver currently handling the device (optional, Linux only).
+
+.TP
+.B Module
+Kernel module reporting that it is capable of handling the device
+(optional, Linux only).
+
+.P
+New tags can be added in future versions, so you should silently ignore any tags you don't recognize.
+
+.SS Backward-compatible verbose format (-vm)
+
+In this mode, lspci tries to be perfectly compatible with its old versions.
+It's almost the same as the regular verbose format, but the
+.B
+Device
+tag is used for both the slot and the device name, so it occurs twice
+in a single record. Please avoid using this format in any new code.
+
+.SH FILES
+.TP
+.B /usr/share/pci.ids
+A list of all known PCI ID's (vendors, devices, classes and subclasses). Maintained
+at http://pciids.sourceforge.net/, use the
+.B update-pciids
+utility to download the most recent version.
+.TP
+.B /usr/share/pci.ids.gz
+If lspci is compiled with support for compression, this file is tried before pci.ids.
+.TP
+.B ~/.pciids-cache
+All ID's found in the DNS query mode are cached in this file.
+
+.SH BUGS
+
+Sometimes, lspci is not able to decode the configuration registers completely.
+This usually happens when not enough documentation was available to the authors.
+In such cases, it at least prints the
+.B <?>
+mark to signal that there is potentially something more to say. If you know
+the details, patches will be of course welcome.
+
+Access to the extended configuration space is currently supported only by the
+.B linux_sysfs
+back-end.
+
+.SH SEE ALSO
+.BR setpci (8),
+.BR update-pciids (8),
+.BR pcilib (7)
+
+.SH AUTHOR
+The PCI Utilities are maintained by Martin Mares <mj@ucw.cz>.
Binary files tmp/pciutils-3.1.2/lspci.o and work/main.d/pciutils-3.1.2/lspci.o differ
Binary files tmp/pciutils-3.1.2/ls-tree.o and work/main.d/pciutils-3.1.2/ls-tree.o differ
Binary files tmp/pciutils-3.1.2/ls-vpd.o and work/main.d/pciutils-3.1.2/ls-vpd.o differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/Makefile work/main.d/pciutils-3.1.2/Makefile
--- tmp/pciutils-3.1.2/Makefile	2009-02-01 13:49:22.000000000 -0300
+++ work/main.d/pciutils-3.1.2/Makefile	2009-04-29 11:19:50.000000000 -0300
@@ -98,6 +98,9 @@
 
 distclean: clean
 
+
+install-all: install install-lib
+
 install: all
 # -c is ignored on Linux, but required on FreeBSD
 	$(DIRINSTALL) -m 755 $(DESTDIR)$(SBINDIR) $(DESTDIR)$(IDSDIR) $(DESTDIR)$(MANDIR)/man8 $(DESTDIR)$(MANDIR)/man7
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/pci.ids work/main.d/pciutils-3.1.2/pci.ids
--- tmp/pciutils-3.1.2/pci.ids	2009-01-30 10:18:39.000000000 -0300
+++ work/main.d/pciutils-3.1.2/pci.ids	2009-04-29 11:20:07.000000000 -0300
@@ -1,8 +1,8 @@
 #
 #	List of PCI ID's
 #
-#	Version: 2009.01.30
-#	Date:    2009-01-30 14:17:35
+#	Version: 2009.04.14
+#	Date:    2009-04-14 03:15:01
 #
 #	Maintained by Martin Mares <mj@ucw.cz> and other volunteers from the
 #	PCI ID Project at http://pciids.sf.net/.
@@ -71,6 +71,7 @@
 02ac  SpeedStream
 	1012  1012 PCMCIA 10/100 Ethernet Card [RTL81xx]
 02e0  XFX Pine Group Inc
+0303  Hewlett-Packard Company (Wrong ID)
 0315  SK-Electronics Co., Ltd.
 0357  TTTech AG
 	000a  TTP-Monitoring Card V2.0
@@ -332,6 +333,7 @@
 		1028 018a  PERC 4/IM
 		1028 1010  LSI U320 SCSI Controller
 		103c 12c5  Ultra320 SCSI [A7173A]
+		103c 1323  Core I/O LAN/SCSI Combo [AB314A]
 		124b 1170  PMC-USCSI320
 		1734 1052  PRIMERGY BX/RX/TX S2 series onboard SCSI(IME)
 	0031  53c1030ZC PCI-X Fusion-MPT Dual Ultra320 SCSI
@@ -352,6 +354,7 @@
 		1028 1f07  SAS 5/iR Integrated RAID Controller
 		1028 1f08  SAS 5/iR Integrated RAID Controller
 		1028 1f09  SAS 5/iR Adapter RAID Controller
+		15ad 1976  SAS Controller
 	0055  SAS1068 PCI-X Fusion-MPT SAS
 		1033 8336  SAS1068
 	0056  SAS1064ET PCI-Express Fusion-MPT SAS
@@ -402,10 +405,15 @@
 	0065  SAS2116 PCI-Express Fusion-MPT SAS-2 [Meteor]
 	0070  SAS2004 PCI-Express Fusion-MPT SAS-2 [Spitfire]
 	0072  SAS2008 PCI-Express Fusion-MPT SAS-2 [Falcon]
+	0073  MegaRAID SAS 9240
+		1000 9240  MegaRAID SAS 9240-4i
 	0074  SAS2108 PCI-Express Fusion-MPT SAS-2 [Liberator]
 	0076  SAS2108 PCI-Express Fusion-MPT SAS-2 [Liberator]
 	0077  SAS2108 PCI-Express Fusion-MPT SAS-2 [Liberator]
 	0079  LSI MegaSAS 9260
+		1000 9262  MegaRAID SAS 9262-8i
+		1000 9263  MegaRAID SAS 9261-8i
+		1000 9281  MegaRAID SAS 9281-8E
 		1028 1f15  PERC H800 Adapter
 		1028 1f16  PERC H700 Adapter
 		1028 1f17  PERC H700 Integrated
@@ -610,12 +618,14 @@
 		103c 308b  MX6125
 		105b 0c81  Realtek ALC 653
 		107b 0300  MX6421
+		1462 0131  MS-1013 Notebook
 	4371  IXP SB400 PCI-PCI Bridge
 		103c 308b  MX6125
 		1462 7217  Aspire L250
 	4372  IXP SB400 SMBus Controller
 		1025 0080  Aspire 5024WLMMi
 		103c 308b  MX6125
+		1462 0131  MS-1013 Notebook
 		1462 7217  Aspire L250
 	4373  IXP SB400 USB2 Host Controller
 		1025 0080  Aspire 5024WLMMi
@@ -631,6 +641,7 @@
 	4376  IXP SB400 IDE Controller
 		1025 0080  Aspire 5024WLMMi
 		103c 308b  MX6125
+		1462 0131  MS-1013 Notebook
 		1462 7217  Aspire L250
 	4377  IXP SB400 PCI-ISA Bridge
 		1025 0080  Aspire 5024WLMi
@@ -639,6 +650,7 @@
 	4378  SB400 AC'97 Modem Controller
 		1025 0080  Aspire 5024WLMMi
 		103c 308b  MX6125
+		1462 0131  MS-1013 Notebook
 	4379  IXP SB400 Serial ATA Controller
 		1462 7141  Aspire L250
 	437a  IXP SB400 Serial ATA Controller
@@ -652,6 +664,7 @@
 		1734 10b8  Realtek High Definition Audio
 	4380  SB600 Non-Raid-5 SATA
 		103c 2813  DC5750 Microtower
+		1458 b005  Gigabyte GA-MA69G-S3H Motherboard
 		1462 7327  K9AG Neo2
 		17f2 5999  KI690-AM2 Motherboard
 	4381  SB600 Raid-5 SATA
@@ -690,6 +703,7 @@
 		17f2 5000  KI690-AM2 Motherboard
 	438c  SB600 IDE
 		103c 280a  DC5750 Microtower
+		1458 5002  Gigabyte GA-MA69G-S3H Motherboard
 		1462 7368  K9AG Neo2
 		17f2 5000  KI690-AM2 Motherboard
 	438d  SB600 PCI to LPC Bridge
@@ -1027,7 +1041,7 @@
 # The IBM card doubles as an ATI PCI video adapter
 		1014 029a  Remote Supervisor Adapter II (RSA2)
 		1014 02c8  eServer xSeries server mainboard
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 Embedded Radeon 7000/VE
 		1028 016d  PowerEdge 2850 Embedded Radeon 7000-M
 		1028 0170  PowerEdge 6850 Embedded Radeon 7000/VE
 		1028 019a  PowerEdge SC1425
@@ -1169,6 +1183,7 @@
 		1025 0080  Aspire 5024WLMMi
 		103c 280a  DC5750 Microtower
 		103c 308b  MX6125
+		1462 0131  MS-1013 Notebook
 		1462 7217  Aspire L250
 	5951  Radeon Xpress 200 (RS480/RS482/RX480/RX482) Chipset - Host bridge
 	5952  RD580 [CrossFire Xpress 3200] Chipset Host Bridge
@@ -1177,6 +1192,7 @@
 	5955  Radeon XPRESS 200M 5955 (PCIE)
 		1002 5955  RS480 0x5955 [Radeon XPRESS 200M 5955 (PCIE)]
 		103c 308b  MX6125
+		1462 0131  MS-1013 Notebook
 	5956  RD790 Northbridge only dual slot PCI-e_GFX and HT3 K8 part
 	5957  RX780/RX790 Chipset Host Bridge
 	5958  RD780 Northbridge only dual slot PCI-e_GFX and HT1 K8 part
@@ -1484,6 +1500,7 @@
 	9443  R700 [Radeon HD 4850]
 	944a  M98L [Mobility Radeon HD 4850]
 	944c  RV770 LE [Radeon HD 4800 Series]
+	9450  RV770 [FireStream 9270]
 	9452  RV770 [FireStream 9250]
 	945a  M98 XT [Mobility Radeon HD 4870]
 	9480  M96 [Mobility Radeon HD 4650]
@@ -1516,6 +1533,7 @@
 	954f  RV710 [Radeon HD 4350]
 	9552  M92 LP [Mobility Radeon HD 4300 Series]
 	9553  M92 [Mobility Radeon HD 4500 Series]
+	9555  M93 [Mobility Radeon HD 4500 Series]
 	9559  Mobility Radeon HD 3600 Series
 	9581  M76 [Radeon Mobility HD 2600 Series]
 	9583  M76XT [Mobility Radeon HD 2600 XT]
@@ -1533,6 +1551,7 @@
 	9596  RV635 PRO AGP [Radeon HD 3650]
 	9598  Mobility Radeon HD 3600 Series
 		1002 9598  Mobility Radeon HD 3600
+		1043 01d6  EAH3650 Silent
 	95c0  Mobility Radeon HD 3470
 		1002 95c0  Mobility Radeon HD 3470
 	95c4  Mobility Radeon HD 3400 Series
@@ -1540,7 +1559,9 @@
 	95c5  Mobility Radeon HD 3450
 		1002 95c5  Mobility Radeon HD 3450
 	95c6  RV620 LE AGP [Radeon HD 3450]
+	95c9  RV620 PCI [Radeon HD 3450]
 	95cc  RV620 [ATI FireGL V3700]
+	95cd  RV620 [FireMV 2450]
 	95ce  RV620 [FirePro 2260]
 	95cf  RV620 [FirePro 2260]
 	960f  RS780 Azalia controller
@@ -1549,6 +1570,7 @@
 	9612  RS780M/RS780MN [Radeon HD 3200 Graphics]
 	9613  RS780MC [Radeon HD 3100 Graphics]
 	9614  Radeon HD 3300 Graphics
+	9615  RS780E [AMD 780E]
 	9616  760G [Radeon 3000]
 	aa00  R600 Audio Device [Radeon HD 2900 Series]
 	aa08  RV630/M76 audio device [Radeon HD 2600 Series]
@@ -2023,16 +2045,16 @@
 	1101  K8 [Athlon64/Opteron] Address Map
 	1102  K8 [Athlon64/Opteron] DRAM Controller
 	1103  K8 [Athlon64/Opteron] Miscellaneous Control
-	1200  Family 10h [Opteron, Athlon64, Sempron] HyperTransport Configuration
-	1201  Family 10h [Opteron, Athlon64, Sempron] Address Map
-	1202  Family 10h [Opteron, Athlon64, Sempron] DRAM Controller
-	1203  Family 10h [Opteron, Athlon64, Sempron] Miscellaneous Control
-	1204  Family 10h [Opteron, Athlon64, Sempron] Link Control
-	1300  Family 11h HyperTransport Configuration
-	1301  Family 11h Address Map
-	1302  Family 11h DRAM Controller
-	1303  Family 11h Miscellaneous Control
-	1304  Family 11h Link Control
+	1200  K10 [Opteron, Athlon64, Sempron] HyperTransport Configuration
+	1201  K10 [Opteron, Athlon64, Sempron] Address Map
+	1202  K10 [Opteron, Athlon64, Sempron] DRAM Controller
+	1203  K10 [Opteron, Athlon64, Sempron] Miscellaneous Control
+	1204  K10 [Opteron, Athlon64, Sempron] Link Control
+	1300  Mobile K10 [Turion X2, Athlon X2, Sempron] HyperTransport Configuration
+	1301  Family 11h [Turion X2, Athlon X2, Sempron] Address Map
+	1302  Mobile K10 [Turion X2, Athlon X2, Sempron] DRAM Controller
+	1303  Mobile K10 [Turion X2, Athlon X2, Sempron] Miscellaneous Control
+	1304  Mobile K10 [Turion X2, Athlon X2, Sempron] Link Control
 	2000  79c970 [PCnet32 LANCE]
 		1014 2000  NetFinity 10/100 Fast Ethernet
 		1022 2000  PCnet - Fast 79C971
@@ -2424,8 +2446,13 @@
 		102b 1030  Parhelia 256 MB Dual DVI
 		102b 14e1  Parhelia PCI 256MB
 		102b 2021  QID Pro
+	0530  MGA G200EV
 	0532  MGA G200eW WPCM450
+		1028 0235  PowerEdge R710 MGA G200eW WPCM450
+		1028 0236  PowerEdge R610 MGA G200eW WPCM450
+		1028 0237  PowerEdge T610 MGA G200eW WPCM450
 		1028 0287  PowerEdge M610 MGA G200eW WPCM450
+		1028 029c  PowerEdge M710 MGA G200eW WPCM450
 	0540  M91XX
 		102b 2080  M9140 LP PCIe x16
 		102b 2100  M9120 PCIe x16
@@ -2702,7 +2729,6 @@
 		1019 0970  P6STP-FL motherboard
 		1043 8035  CUSI-FX motherboard
 	6306  530/620 PCI/AGP VGA Display Adapter
-		1039 6306  SiS530,620 GUI Accelerator+3D
 	6325  65x/M650/740 PCI/AGP VGA Display Adapter
 		1039 6325  SiS 651 onboard [Asus P4SC-EA]
 		1631 1004  SiS 651C onboard [Gigabyte GA-8SIML Rev1.0]
@@ -2712,6 +2738,7 @@
 		1092 0a70  SpeedStar A70
 		1092 4910  SpeedStar A70
 		1092 4920  SpeedStar A70
+		10b0 6326  S6110-B (AGP)
 		1569 6326  SiS6326 GUI Accelerator
 	6330  661/741/760 PCI/AGP or 662/761Gx PCIE VGA Display Adapter
 		1039 6330  [M]661xX/[M]741[GX]/[M]760 PCI/AGP VGA Adapter
@@ -2907,6 +2934,7 @@
 # Found on ASUS M2V motherboard
 	81e7  Realtek ALC-660 6-channel CODEC
 	81f4  EN7300TC512/TD/128M/A(C262G) [Graphics Card EN7300TC512]
+	82e8  M3N72-D
 1044  Adaptec (formerly DPT)
 	1012  Domino RAID Engine
 	a400  SmartCache/Raid I-IV Controller
@@ -3401,7 +3429,8 @@
 	6400  MPC190 Security Processor (S1 family, encryption)
 	6405  MPC184 Security Processor (S1 family)
 1058  Electronics & Telecommunications RSH
-1059  Teknor Industrial Computers Inc
+# Formerly: Teknor Industrial Computers Inc
+1059  Kontron
 105a  Promise Technology, Inc.
 	0d30  PDC20265 (FastTrak100 Lite/Ultra100)
 		1043 8042  AV7266-E South Bridge Promise RAID
@@ -4349,6 +4378,7 @@
 		10b5 3353  Alpermann+Velte PCL PCIe D: Timecode Reader Board
 		10b5 3354  Alpermann+Velte PCL PCIe LV: Timecode Reader Board
 		10b5 3355  Alpermann+Velte PCL PCIe L: Timecode Reader Board
+		10b5 3415  Alpermann+Velte PCIe TS: Time Synchronisation Board
 		1369 c001  LX6464ES
 		1369 c201  LX1616ES
 		14b4 d140  Dektec DTA-140
@@ -5081,20 +5111,24 @@
 		10de 0c11  Winfast NF3250K8AA
 		1462 7030  K8N Neo-FSR v2.0
 		147b 1c0b  NF8 Mainboard
+		1849 00e0  Motherboard (one of many)
 	00e1  nForce3 250Gb Host Bridge
 		1043 813f  K8N-E
 		1462 7030  K8N Neo-FSR v2.0
 		147b 1c0b  NF8 Mainboard
+		1849 00e1  Motherboard (one of many)
 	00e2  nForce3 250Gb AGP Host to PCI Bridge
 	00e3  nForce3 Serial ATA Controller
 		1043 813f  K8N-E
 		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
+		1849 00e3  Motherboard (one of many)
 	00e4  nForce 250Gb PCI System Management
 		1043 813f  K8N-E
 		105b 0c43  Winfast NF3250K8AA
 		1462 7030  K8N Neo-FSR v2.0
 		147b 1c0b  NF8 Mainboard
+		1849 00e4  Motherboard (one of many)
 	00e5  CK8S Parallel ATA Controller (v2.5)
 		1043 813f  K8N-E
 		105b 0c43  Winfast NF3250K8AA
@@ -5107,11 +5141,13 @@
 		105b 0c43  Winfast NF3250K8AA
 		1462 7030  K8N Neo-FSR v2.0
 		147b 1c0b  NF8 Mainboard
+		1849 00e7  Motherboard (one of many)
 	00e8  nForce3 EHCI USB 2.0 Controller
 		1043 813f  K8N-E
 		105b 0c43  Winfast NF3250K8AA
 		1462 7030  K8N Neo-FSR v2.0
 		147b 1c0b  NF8 Mainboard
+		1849 00e8  Motherboard (one of many)
 	00ea  nForce3 250Gb AC'97 Audio Controller
 		1043 819d  K8N-E
 		105b 0c43  Winfast NF3250K8AA
@@ -5297,7 +5333,7 @@
 	01d1  G72 [GeForce 7300 LE]
 		1462 0345  7300LE PCI Express Graphics Adapter
 	01d3  G72 [GeForce 7300 SE]
-	01d6  GeForce Go 7200
+	01d6  G72M [GeForce Go 7200]
 	01d7  G72M [Quadro NVS 110M/GeForce Go 7300]
 	01d8  G72M [GeForce Go 7400]
 		1028 01d7  XPS M1210
@@ -5466,8 +5502,8 @@
 		1043 8225  GeForce 7950 GT
 		107d 2a68  WinFast PX7950GT TDH
 		1462 0663  NX7950GT-VT2D512EZ-HD
-	0297  GeForce Go 7950 GTX
-	0298  GeForce Go 7900 GS
+	0297  G71 [GeForce Go 7950 GTX]
+	0298  G71 [GeForce Go 7900 GS]
 	0299  GeForce Go 7900 GTX
 	029a  G71 [Quadro FX 2500M]
 	029b  G71 [Quadro FX 1500M]
@@ -5479,15 +5515,14 @@
 	02a0  NV2A [XGPU]
 	02a5  MCPX CPU Bridge
 	02a6  MCPX Memory Controller
-	02e0  GeForce 7600 GT
+	02e0  G73 [GeForce 7600 GT]
 		02e0 2249  GF 7600GT 560M 256MB DDR3 DUAL DVI TV
 	02e1  G73 [GeForce 7600 GS]
 		1682 222b  PV-T73K-UAL3 (256MB)
 		1682 2247  GF 7600GS 512MB DDR2
-	02e2  GeForce 7300 GT
-	02e3  GeForce 7900 GS
-# An oddball 7950 that doesn't show up on any official NVIDIA lists but was seen in the wild
-	02e4  GeForce 7950 GT AGP
+	02e2  G73 [GeForce 7300 GT]
+	02e3  G71 [GeForce 7900 GS]
+	02e4  G71 [GeForce 7950 GT]
 		1682 2271  PV-T71A-YDF7 (512MB)
 	02f0  C51 Host Bridge
 		103c 30b7  Presario V6133CL
@@ -5654,11 +5689,11 @@
 		1462 0412  NX7300GT-TD256EH
 	0394  G70 [GeForce 7600 LE]
 	0395  G70 [GeForce 7300 GT]
-	0397  GeForce Go 7700
-	0398  G70 [GeForce Go 7600]
+	0397  G73 [GeForce Go 7700]
+	0398  G73 [GeForce Go 7600]
 		1025 006c  Acer 9814 WKMI
-	0399  GeForce Go 7600 GT
-	039b  GeForce Go 7900 SE
+	0399  G73 [GeForce Go 7600 GT]
+	039b  G73 [GeForce Go 7900 SE]
 	039c  Quadro FX 550M
 		10de 039c  Quadro FX 560M
 	039e  G73GL [Quadro FX 560]
@@ -5738,10 +5773,10 @@
 	0403  GeForce 8600 GS
 	0404  GeForce 8400 GS
 		1462 1230  NX8400GS-TD256E
-	0405  GeForce 9500M GS
-	0407  GeForce 8600M GT
-	0408  GeForce 9650M GS
-	0409  GeForce 8700M GT
+	0405  G84 [GeForce 9500M GS]
+	0407  G84 [GeForce 8600M GT]
+	0408  G84 [GeForce 9650M GS]
+	0409  G84 [GeForce 8700M GT]
 	040a  Quadro FX 370
 	040b  Quadro NVS 320M
 	040c  Quadro FX 570M
@@ -5754,17 +5789,17 @@
 	0422  GeForce 8400 GS
 	0423  GeForce 8300 GS
 	0424  GeForce 8400 GS
-	0425  GeForce 8600M GS
+	0425  G86 [GeForce 8600M GS]
 		1025 0121  Aspire 5920G
-	0426  G86M [GeForce 8400M GT]
-	0427  GeForce 8400M GS
-	0428  GeForce 8400M G
+	0426  G86 [GeForce 8400M GT]
+	0427  G86 [GeForce 8400M GS]
+	0428  G86 [GeForce 8400M G]
 	0429  Quadro NVS 140M
 		17aa 20d8  ThinkPad T61
 	042a  Quadro NVS 130M
 	042b  Quadro NVS 135M
 	042d  Quadro FX 360M
-	042e  GeForce 9300M G
+	042e  G86 [GeForce 9300M G]
 	042f  Quadro NVS 290
 	0440  MCP65 LPC Bridge
 	0441  MCP65 LPC Bridge
@@ -5797,38 +5832,54 @@
 	045d  MCP65 SATA Controller
 	045e  MCP65 SATA Controller
 	045f  MCP65 SATA Controller
-	0531  GeForce 7150M
-	0533  GeForce 7000M (rev a2)
+	0531  C67 [GeForce 7150M / nForce 630M]
+	0533  C67 [GeForce 7000M / nForce 610M]
 	053a  GeForce 7050 PV / nForce 630a
 	053b  GeForce 7050 PV / nForce 630a
+		1043 8308  M2N68-AM Motherbord
 	053e  GeForce 7025
 	0541  MCP67 Memory Controller
 	0542  MCP67 SMBus
+		1043 8308  M2N68-AM Motherbord
 	0543  MCP67 Co-processor
 	0547  MCP67 Memory Controller
+		1043 8308  M2N68-AM Motherbord
 		1849 0547  ALiveNF7G-HDready
 	0548  MCP67 ISA Bridge
+		1043 8308  M2N68-AM Motherboard
 	054c  MCP67 Ethernet
+		1043 8308  M2N68-AM Motherbord
 		1849 054c  ALiveNF7G-HDready, MCP67 Gigabit Ethernet
 	054d  MCP67 Ethernet
 	054e  MCP67 Ethernet
 # forcedeth source code from Nvidi
 	054f  MCP67 Ethernet
 	0550  MCP67 AHCI Controller
+		1043 8308  M2N68-AM Motherboard
 	0554  MCP67 AHCI Controller
+		1043 8308  M2N68-AM Motherboard
 	0555  MCP67 SATA Controller
+		1043 8308  M2N68-AM Motherboard
 	055c  MCP67 High Definition Audio
+		1043 8290  M2N68-AM Motherboard
 	055d  MCP67 High Definition Audio
 	055e  MCP67 OHCI USB 1.1 Controller
+		1043 8308  M2N68-AM Motherboard
 	055f  MCP67 EHCI USB 2.0 Controller
+		1043 8308  M2N68-AM Motherboard
 	0560  MCP67 IDE Controller
+		f043 8308  M2N68-AM Motherboard
 	0561  MCP67 PCI Bridge
 	0562  MCP67 PCI Express Bridge
 		1849 0562  ALiveNF7G-HDready
 	0563  MCP67 PCI Express Bridge
 	0568  MCP78S [GeForce 8200] Memory Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0568  K10N78FullHD-hSLI R3.0 Memory Controller
 	0569  MCP78S [GeForce 8200] PCI Express Bridge
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0569  K10N78FullHD-hSLI R3.0 PCI Express Bridge
 	056a  MCP73 [nForce 630i] USB 2.0 Controller (EHCI)
 	056c  MCP73 IDE
@@ -5836,88 +5887,156 @@
 	056d  MCP73 PCI Express bridge
 	056e  MCP73 PCI Express bridge
 	056f  MCP73 PCI Express bridge
+	05b8  PCI express bridge for GTX 295
 	05be  PCI express bridge for Quadro Plex S4 / Tesla S870 / Tesla S1070
-	05e1  GT200 [GTX280]
-	05e2  GT200 [GTX260-216]
+	05e0  GT200b [GeForce GTX 295]
+	05e1  GT200 [GeForce GTX 280]
+# GT200 [GTX 260] or GT200 [GTX 260-216] or GT200b [GTX 260-216]
+	05e2  GT200 [GeForce GTX 260]
+	05e3  GT200b [GeForce GTX 285]
+	05e6  GT200b [GeForce GTX 275]
 	05e7  GT200 [Tesla C1060 / Tesla S1070]
-	05f9  Quadro CX
-	05fd  Quadro FX 5800
-	05fe  Quadro FX 4800
+	05ed  GT200GL [Quadro Plex 2200 D2]
+	05f8  GT200GL [Quadro Plex 2200 S4]
+	05f9  GT200GL [Quadro CX]
+	05fd  GT200GL [Quadro FX 5800]
+	05fe  GT200GL [Quadro FX 4800]
 	0600  GeForce 8800 GTS 512
 	0601  GeForce 9800 GT 512
 	0602  GeForce 8800 GT 512
 	0604  GeForce 9800 GX2
 	0605  GeForce 9800 GT
 	0606  GeForce 8800 GS
-	0609  GeForce 8800M GTS
-	060c  GeForce 8800M GTX
+	0608  G92 [GeForce 9800M GTX]
+	0609  G92 [GeForce 8800M GTS]
+	060b  G92 [GeForce 9800M GT]
+	060c  G92 [GeForce 8800M GTX]
 	060d  GeForce 8800 GS
 	0610  GeForce 9600 GSO
+		1682 2385  GeForce 9600 GSO 768mb
 	0611  GeForce 8800 GT
 		107d 2ab0  Winfast PX8800 GT PCI-E
 		19da 1040  ZT-88TES2P-FSP
 	0612  GeForce 9800 GTX
 	0613  GeForce 9800 GTX+
 	0614  GeForce 9800 GT
-	0617  GeForce 9800M GTX
-	061a  Quadro FX 3700
-	061c  Quadro FX 3600M
+		107d 2ab3  WinFast PX9800 GT (S-Fanpipe)
+	0615  G92 [GeForce GTS 250]
+		3842 1150  GeForce GTS 250 P/N 512-P3-1150-TR
+# Overclocked
+		3842 1151  GeForce GTS 250 P/N 512-P3-1151-TR
+		3842 1155  GeForce GTS 250 P/N 01G-P3-1155-TR
+# Overclocked
+		3842 1156  GeForce GTS 250 P/N 01G-P3-1156-TR
+	0617  G92 [GeForce 9800M GTX]
+	0619  G92GL [Quadro FX 4700 X2]
+	061a  G92 [Quadro FX 3700]
+	061b  G92GL [Quadro VX 200]
+	061c  G92M [Quadro FX 3600M]
 	0622  G94 [GeForce 9600 GT]
+		107d 2ac1  WinFast PX9600GT 1024MB
 		1458 3481  GV-NX96T512HP
 	0623  GeForce 9600 GS
-	063a  Quadro FX 2700M
+	0625  GeForce 9600 GSO
+	0628  G94 [GeForce 9800M GTS]
+	062a  G94 [GeForce 9700M GTS]
+	062b  G94 [GeForce 9800M GS]
+	062c  G94 [GeForce 9800M GTS]
+	0638  G94 [Quadro FX 1800]
+	063a  G94M [Quadro FX 2700M]
 	0640  GeForce 9500 GT
-	0641  D9M-20 [GeForce 9400 GT]
+	0641  G96 [GeForce 9400 GT]
 	0647  GeForce 9600M GT
-	0648  GeForce 9600M GS
-	0649  GeForce 9600M GT
-	064b  GeForce 9500M G
-	065c  Quadro FX 770M
+	0648  G96 [GeForce 9600M GS]
+	0649  G96 [GeForce 9600M GT]
+	064a  G96 [GeForce 9700M GT]
+	064b  G96 [GeForce 9500M G]
+	064c  G96 [GeForce 9650M GT]
+	0651  G96 [GeForce G 110M]
+	0652  G96 [GeForce GT 130M]
+	0658  G96 [Quadro FX 380]
+	0659  G96 [Quadro FX 580]
+	065c  G96M [Quadro FX 770M]
 	06e2  GeForce 8400 GS
 	06e4  GeForce 8400 GS
 		1458 3475  GV-NX84S256HE [GeForce 8400 GS]
 	06e5  GeForce 9300M GS
-	06e8  GeForce 9200M GS
-	06e9  GeForce 9300M GS
+# G98 [GeForce 9200M GE]
+	06e8  G98 [GeForce 9200M GS]
+	06e9  G98 [GeForce 9300M GS]
 		1043 19b2  U6V laptop
-	06ea  Quadro NVS 150M
-	06eb  Quadro NVS 160M
-	06f9  Quadro FX 370 LP
-	06fa  Quadro NVS 450
+	06ea  G86M [Quadro NVS 150M]
+	06eb  G98M [Quadro NVS 160M]
+	06ec  G98 [G 105M]
+	06f9  G98 [Quadro FX 370 LP]
+	06fa  G98 [Quadro NVS 450]
 	0751  MCP78S [GeForce 8200] Memory Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0751  K10N78FullHD-hSLI R3.0 Memory Controller
 	0752  MCP78S [GeForce 8200] SMBus
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0752  K10N78FullHD-hSLI R3.0 SMBus
 	0753  MCP78S [GeForce 8200] Co-Processor
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0753  K10N78FullHD-hSLI R3.0 Co-Processor
 	0754  MCP78S [GeForce 8200] Memory Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0754  K10N78FullHD-hSLI R3.0 Memory Controller
 	0759  MCP78S [GeForce 8200] IDE
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0759  K10N78FullHD-hSLI R3.0 IDE
 	075a  MCP78S [GeForce 8200] PCI Bridge
+		1043 82e8  M3N72-D
 		1849 075a  K10N78FullHD-hSLI R3.0 PCI Bridge
 	075b  MCP78S [GeForce 8200] PCI Express Bridge
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 075b  K10N78FullHD-hSLI R3.0 PCI Express Bridge
 	075c  MCP78S [GeForce 8200] LPC Bridge
+		1462 7508  K9N2GM-FIH
 		1849 075c  K10N78FullHD-hSLI R3.0 LPC Bridge
-	0760  MCP78S [GeForce 8200] Ethernet
+	075d  nForce 750a LPC Bridge
+		1043 82e8  M3N72-D
+	0760  MCP77 Ethernet
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0760  K10N78FullHD-hSLI R3.0 Ethernet
 	0761  MCP77 Ethernet
 	0762  MCP77 Ethernet
 	0763  MCP77 Ethernet
-	0774  MCP78S [GeForce 8200] High Definition Audio
+	0774  MCP72XE/MCP72P/MCP78U/MCP78S High Definition Audio
+# has a Realtek ALC1200 HDAudio Codec
+		1043 82fe  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 3662  K10N78FullHD-hSLI R3.0 High Definition Audio
 	0778  MCP78S [GeForce 8200] PCI Express Bridge
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 0778  K10N78FullHD-hSLI R3.0 PCI Express Bridge
 	077a  MCP78S [GeForce 8200] PCI Bridge
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 077a  K10N78FullHD-hSLI R3.0 PCI Bridge
 	077b  MCP78S [GeForce 8200] OHCI USB 1.1 Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 077b  K10N78FullHD-hSLI R3.0 OHCI USB 1.1 Controller
 	077c  MCP78S [GeForce 8200] EHCI USB 2.0 Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 077c  K10N78FullHD-hSLI R3.0 EHCI USB 2.0 Controller
 	077d  MCP78S [GeForce 8200] OHCI USB 1.1 Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 077d  K10N78FullHD-hSLI R3.0 OHCI USB 1.1 Controller
 	077e  MCP78S [GeForce 8200] EHCI USB 2.0 Controller
+		1043 82e8  M3N72-D
+		1462 7508  K9N2GM-FIH
 		1849 077e  K10N78FullHD-hSLI R3.0 EHCI USB 2.0 Controller
 	07c0  MCP73 Host Bridge
 		1afa 7150  JW-IN7150-HD
@@ -5969,29 +6088,31 @@
 		10de 07fc  MCP73 High Definition Audio
 	07fe  GeForce 7100/nForce 630i
 		1afa 7150  JW-IN7150-HD
-	0844  GeForce 9100M G
-	0845  GeForce 8200M G
+	0844  C77 [GeForce 9100M G]
+	0845  C77 [GeForce 8200M G]
 	0848  GeForce 8300
 	0849  GeForce 8200
+		1462 7508  K9N2GM-FIH
 		1849 0849  K10N78FullHD-hSLI R3.0 GeForce 8200
 	084a  nForce 730a
 	084b  GeForce 8200
 	084c  nForce 780a SLI
-	084d  nForce 750a SLI
+	084d  nForce 750a SLI [mGPU]
+		1043 82e8  M3N72-D mGPU
 	084f  GeForce 8100 / nForce 720a
 	0860  GeForce 9300
 	0861  GeForce 9400
 	0862  GeForce 9400M G
-	0863  GeForce 9400M
+	0863  C79 [GeForce 9400M]
 	0864  GeForce 9300
 	0865  GeForce 9300
-	0866  GeForce 9400M G
+	0866  C79 [GeForce 9400M G]
 	0867  GeForce 9400
 	086a  GeForce 9400
 	086c  GeForce 9300 / nForce 730i
-	086e  GeForce 9100M G
-	086f  GeForce 9200M G
-	0870  GeForce 9400M
+	086e  C79 [GeForce 9100M G]
+	086f  C79 [GeForce 9200M G]
+	0870  C79 [GeForce 9400M]
 	087a  Quadro FX 470
 	0a80  MCP79 Host Bridge
 	0a81  MCP79 Host Bridge
@@ -6044,12 +6165,11 @@
 	0ac7  MCP79 PCI Express Bridge
 	0ac8  MCP79 PCI Express Bridge
 	0ad0  MCP78S [GeForce 8200] SATA Controller (non-AHCI mode)
+		1462 7508  K9N2GM-FIH
 		1849 0ad0  K10N78FullHD-hSLI R3.0 IDE
 	0ad4  MCP78S [GeForce 8200] AHCI Controller
+		1043 82e8  M3N72-D
 		1849 0ad4  K10N78FullHD-hSLI R3.0 AHCI Controller
-	10de  GeForce 9800M GTX
-	21c3  nVidia Corporation G73 [GeForce 7600 GS] (rev a2)
-	c615  G70 [GeForce 7600 GT]
 10df  Emulex Corporation
 	1ae5  LP6000 Fibre Channel Host Adapter
 	e100  Proteus-X: LightPulse IOV Fibre Channel Host Adapter
@@ -6068,6 +6188,8 @@
 	f0e5  Zephyr LightPulse Fibre Channel Host Adapter
 	f0f5  Neptune LightPulse Fibre Channel Host Adapter
 	f100  Saturn-X: LightPulse Fibre Channel Host Adapter
+	f111  Saturn-X LightPulse Fibre Channel Host Adapter
+	f112  Saturn-X LightPulse Fibre Channel Host Adapter
 	f700  LP7000 Fibre Channel Host Adapter
 	f701  LP7000 Fibre Channel Host Adapter Alternate ID (JX1:2-3, JX2:1-2)
 	f800  LP8000 Fibre Channel Host Adapter
@@ -6086,8 +6208,12 @@
 	fc40  Saturn-X: LightPulse Fibre Channel Host Adapter
 	fc50  Proteus-X: LightPulse IOV Fibre Channel Host Adapter
 	fd00  Helios-X LightPulse Fibre Channel Host Adapter
+	fd11  Helios-X LightPulse Fibre Channel Host Adapter
+	fd12  Helios-X LightPulse Fibre Channel Host Adapter
 	fe00  Zephyr-X LightPulse Fibre Channel Host Adapter
 	fe05  Zephyr-X: LightPulse FCoE Adapter
+	fe11  Zephyr-X LightPulse Fibre Channel Host Adapter
+	fe12  Zephyr-X LightPulse FCoE Adapter
 	ff00  Neptune LightPulse Fibre Channel Host Adapter
 10e0  Integrated Micro Solutions Inc.
 	5026  IMS5026/27/28
@@ -6185,6 +6311,7 @@
 		1113 1208  EN1208
 		1186 0300  DE-528
 		1259 2400  AT-2400
+		1af4 1100  Qemu virtual machine
 	8129  RTL-8129
 		10ec 8129  RT8129 Fast Ethernet Adapter
 		11ec 8129  RT8129 Fast Ethernet Adapter
@@ -6203,7 +6330,6 @@
 		1043 8109  P5P800-MX Mainboard
 		1071 8160  MIM2000
 		10bd 0320  EP-320X-R
-		10ec 8139  RT8139
 		10f7 8338  Panasonic CF-Y5 laptop
 		1113 ec01  FNC-0107TX
 		1186 1300  DFE-538TX
@@ -6219,6 +6345,7 @@
 		1436 8139  RT8139
 		144d c00c  P30/P35 notebook
 		1458 e000  GA-7VM400M/7VT600 Motherboard
+		1462 0131  MS-1013 Notebook
 		1462 217c  Aspire L250
 		1462 788c  865PE Neo2-V Mainboard
 		146c 1439  FE-1439TX
@@ -6240,6 +6367,7 @@
 		8e2e 7100  KF-230TX/2
 		a0a0 0007  ALN-325C
 	8167  RTL-8110SC/8169SC Gigabit Ethernet
+		1458 e000  GA-MA69G-S3H Motherboard
 		1462 235c  P965 Neo MS-7235 mainboard
 		1462 236c  945P Neo3-F motherboard
 	8168  RTL8111/8168B PCI Express Gigabit Ethernet controller
@@ -6249,9 +6377,10 @@
 		1458 e000  GA-EP45-DS5 Motherboard
 		1462 238c  Onboard RTL8111b on MSI P965 Platinum Mainboard
 		1462 368c  K9AG Neo2
-		1849 8168  A770CrossFire motherboard
+		1849 8168  Motherboard (one of many)
 	8169  RTL-8169 Gigabit Ethernet
 		1025 0079  Aspire 5024WLMi
+		10bd 3202  EP-320G-TX1 32-bit PCI Gigabit Ethernet Adapter
 		1259 c107  CG-LAPCIGT
 		1371 434e  ProG-2000L
 		1385 311a  GA311
@@ -6377,19 +6506,19 @@
 		1102 4004  EMU1010 Digital Audio System [MAEM8960]
 	0009  [SB X-Fi Xtreme Audio] CA0110-IBG
 		1102 0010  [SB X-Fi Xtreme Audio] CA0110-IBG
-	000b  [X-Fi Titanium series] EMU20k2
+	000b  X-Fi Titanium series [EMU20k2]
+		1102 0041  SB X-Fi Titanium PCI-e [SB0880]
 	4001  SB Audigy FireWire Port
-		1102 0010  EMU 1010
+		1102 0010  SB Audigy FireWire Port
 	7002  SB Live! Game Port
 		1102 0020  Gameport Joystick
 	7003  SB Audigy Game Port
-		1102 0040  SB Audigy MIDI/Game Port
+		1102 0040  SB Audigy Game Port
 		1102 0060  SB Audigy2 MIDI/Game Port
 	7004  [SB Live! Value] Input device controller
 	7005  SB Audigy LS Game Port
 		1102 1001  SB0310 Audigy LS MIDI/Game port
 		1102 1002  SB0312 Audigy LS MIDI/Game port
-	8064  SB0100 [SBLive! 5.1 OEM]
 	8938  Ectiva EV1938
 		1033 80e5  SlimTower-Jim (NEC)
 		1071 7150  Mitac 7150
@@ -6517,10 +6646,12 @@
 	0598  VT82C598 [Apollo MVP3]
 	0601  VT8601 [Apollo ProMedia]
 	0605  VT8605 [ProSavage PM133]
+		103c 1254  D9840-60001 [Brio BA410 Motherboard]
 		1043 802c  CUV4X mainboard
 	0680  VT82C680 [Apollo P6]
 	0686  VT82C686 [Apollo Super South]
 		1019 0985  P6VXA Motherboard
+		103c 1256  D9840-60001 [Brio BA410 Motherboard]
 		1043 802c  CUV4X mainboard
 		1043 8033  A7V Mainboard
 		1043 803e  A7V-E Mainboard
@@ -6587,6 +6718,7 @@
 	287e  VT8251 Ultra VLINK Controller
 	3022  CLE266
 	3038  VT82xxxxx UHCI USB 1.1 Controller
+# possibly Hewlett-Packard D9840-60001 [Brio BA410 Motherboard]
 		0925 1234  VA-502 Mainboard
 		1019 0985  P6VXA Motherboard
 		1019 0a81  L7VTA v1.0 Motherboard (KT400-8235)
@@ -6635,6 +6767,7 @@
 		0e11 b194  Soundmax integrated digital audio
 		1019 0985  P6VXA Motherboard
 		1019 0987  K7VZA Motherboard
+		103c 1251  D9840-60001 [Brio BA410 Motherboard]
 		1043 1106  A7V133/A7V133-C Mainboard
 		1106 4511  Onboard Audio on EP7KXA
 		1106 aa03  VT1612A AC'97 Audio Controller
@@ -7191,10 +7324,23 @@
 	3400  SmartPCI56(UCB1500) 56K Modem
 	5400  TriMedia TM1000/1100
 		12ca 0000  BlueICE
-	5402  TriMedia TM-1300
+	5402  TriMedia TM1300
 		1244 0f00  Fritz!Card DSL
+		15eb 1300  DT1300
+		15eb 1302  DT1302
+		15eb 1304  DT1304
+		15eb 1305  DT1305
+		15eb 1306  PMCDT1306
+		15eb 1308  DT1308
+		15eb 1331  DT1301 with SAA7121
+		15eb 1337  DT1301 with SAA7127
+		15eb 2d3d  X3D
+		15eb 7022  PTM1300
 	5405  TriMedia TM1500
+		1136 0005  LCP-1500
 	5406  TriMedia TM1700
+	540b  PNX1005 Media Processor
+		1131 0020  PNXLite PCI Demo Board
 	7130  SAA7130 Video Broadcast Decoder
 		102b 48d0  Matrox CronosPlus
 		1048 226b  ELSA EX-VISION 300TV
@@ -7233,7 +7379,9 @@
 		1435 7350  VFG7350
 		1461 1044  AVerTVHD MCE A180
 		1461 4836  M10D Hybrid DVBT
+		1461 861e  M105 PAL/SECAM/NTSC/FM Tuner
 		1461 a14b  AVerTV Studio 509
+		1461 a836  M115 DVB-T, PAL/SECAM/NTSC Tuner
 		1461 f01d  DVB-T Super 007
 		1461 f31f  Avermedia AVerTV GO 007 FM
 		1461 f936  Hybrid+FM PCI (rev A16D)
@@ -7319,6 +7467,7 @@
 		153b 1156  Terratec Cynergy 1200C
 		153b 1157  Cinergy 1200 DVB-T
 		1894 0020  KNC One DVB-C V1.0
+		1894 0023  TVStation DVB-C plus
 	7162  Pinnacle PCTV 3010iX Dual Analog + DVB-T (VT8251 Ultra VLINK Controller)
 	9730  SAA9730 Integrated Multimedia and Peripheral Controller
 		1131 0000  Integrated Multimedia and Peripheral Controller
@@ -7882,6 +8031,7 @@
 		1043 1237  A6J-Q008
 		1043 1967  V6800V
 		144d c018  X20 IV
+		17aa 20ca  ThinkPad T61
 	0811  R5C811
 	0822  R5C822 SD/SDIO/MMC/MS/MSPro Host Adapter
 		1014 0556  ThinkPad X60s / Z60t
@@ -7899,6 +8049,7 @@
 		10f7 8338  Panasonic CF-Y5 laptop
 		144d c018  X20 IV
 		17aa 201d  ThinkPad X60s
+		17aa 20c7  ThinkPad T61
 	0832  R5C832 IEEE 1394 Controller
 		1025 0121  Aspire 5920G
 		1028 01d7  XPS M1210
@@ -7922,6 +8073,7 @@
 		103c 30b7  Presario V6133CL
 		1043 1967  V6800V
 		1180 0852  Pavilion 2410us
+		1324 10cf  P7120
 1181  Telmatics International
 1183  Fujikura Ltd
 1184  Forks Inc
@@ -8397,7 +8549,8 @@
 	5801  USB
 	5802  USS-312 USB Controller
 	5803  USS-344S USB Controller
-	5811  FW323
+	5811  FW322/323
+		1043 8294  IEEE 1394a Firewire Controller
 		8086 524c  D865PERL mainboard
 		dead 0800  FireWire Host Bus Adapter
 	5901  FW643 PCI Express1394b Controller (PHY/Link)
@@ -9189,6 +9342,8 @@
 	9009  Ethernet 100/10 MBit
 	9100  21x4x DEC-Tulip compatible 10/100 Ethernet
 	9102  21x4x DEC-Tulip compatible 10/100 Ethernet
+# Subsystem ID is main ID reveresed.
+		0291 8212  DM9102A (DM9102AE, SM9102AF) Ethernet 100/10 MBit
 	9132  Ethernet 100/10 MBit
 1283  Integrated Technology Express, Inc.
 	673a  IT8330G
@@ -9787,6 +9942,9 @@
 	0302  TCR167PCI IRIG Timecode Reader
 	0303  TCR511PCI IRIG Timecode Reader
 	0304  TCR511PEX IRIG Timecode Reader (PCI Express)
+	0305  TCR170PEX IRIG Timecode Reader (PCI Express)
+	0501  PTP270PEX PTP/IEEE1588 slave card (PCI Express)
+	0601  FRC511PEX Free Running Clock (PCI Express)
 1361  Soliton Systems K.K.
 1362  Fujifacom Corporation
 1363  Phoenix Technology Ltd
@@ -9952,7 +10110,7 @@
 		1397 b520  HFC-4S [IOB4ST]
 		1397 b540  HFC-4S [Swyx 4xS0 SX2 QuadBri]
 		1397 b556  HFC-4S [Junghanns DuoDBRI]
-		1397 e888  HFC-4S [OpenVox B400P]
+		1397 e888  HFC-4S [OpenVox B200P / B400P]
 	16b8  ISDN network Controller [HFC-8S]
 	2bd0  ISDN network controller [HFC-PCI]
 		0675 1704  ISDN Adapter (PCI Bus, D, C)
@@ -10028,6 +10186,9 @@
 	0010  SyncLink Adapter v1
 	0020  SyncLink SCC Adapter
 	0030  SyncLink Multiport Adapter
+	0070  SyncLink GT Adapter
+	0080  SyncLink GT4 Adapter
+	00a0  SyncLink GT2 Adapter
 	0210  SyncLink Adapter v2
 13c1  3ware Inc
 	1000  5xxx/6xxx-series PATA-RAID
@@ -10152,6 +10313,7 @@
 		1043 82b7  Virtuoso 200 (Xonar D2X)
 		1043 8314  Virtuoso 200 (Xonar HDAV1.3)
 		1043 834f  Virtuoso 100 (Xonar D1)
+		1043 835c  Virtuoso 100 (Xonar Essence STX)
 		14c3 1710  HIFIER
 		1a58 0910  Barracuda AC-1
 		415a 5431  X-Meridian 7.1
@@ -10371,6 +10533,7 @@
 		131f 2051  CyberSerial 4S Plus
 		15ed 2000  MCCR Serial p0-3 of 8
 		15ed 2001  MCCR Serial p0-3 of 16
+	9505  16950 Compatible (dual) serial controller
 	950a  EXSYS EX-41092 Dual 16950 Serial adapter
 	950b  OXCB950 Cardbus 16950 UART
 	9510  OX16PCI954 (Quad 16950 UART) function 1 (Disabled)
@@ -10750,11 +10913,15 @@
 		107b 5048  E4500 Onboard
 	1601  NetXtreme BCM5752M Gigabit Ethernet PCI Express
 	1639  NetXtreme II BCM5709 Gigabit Ethernet
+		1028 0235  PowerEdge R710 BCM5709 Gigabit Ethernet
+		1028 0236  PowerEdge R610 BCM5709 Gigabit Ethernet
+		1028 0237  PowerEdge T610 BCM5709 Gigabit Ethernet
 		103c 7055  NC382i Integrated Quad Port PCI Express Gigabit Server Adapter
 		103c 7059  NC382T PCI Express Dual Port Multifunction Gigabit Server Adapter
 		10a9 8027  Quad port Gigabit Ethernet Controller
 	163a  NetXtreme II BCM5709S Gigabit Ethernet
 		1028 027b  PowerEdge M610 Broadcom NetXtreme II BCM5709S
+		1028 029c  PowerEdge M710 BCM5709S Gigabit Ethernet
 		103c 171d  NC382m Dual Port 1GbE Multifunction BL-c Adapter
 		103c 7056  NC382i Integrated Quad Port PCI Express Gigabit Server Adapter
 	163b  NetXtreme II BCM5716 Gigabit Ethernet
@@ -10934,9 +11101,12 @@
 	1680  NetXtreme BCM5761e Gigabit Ethernet PCIe
 	1681  NetXtreme BCM5761 Gigabit Ethernet PCIe
 	1684  NetXtreme BCM5764M Gigabit Ethernet PCIe
+	1690  NetXtreme BCM57760 Gigabit Ethernet PCIe
+	1692  NetLink BCM57780 Gigabit Ethernet PCIe
 	1693  NetLink BCM5787M Gigabit Ethernet PCI Express
 		1025 0121  Aspire 5920G
 		103c 30c0  6710b
+	1694  NetLink BCM57790 Gigabit Ethernet PCIe
 	1696  NetXtreme BCM5782 Gigabit Ethernet
 		103c 12bc  d530 CMT (DG746A)
 		14e4 000d  NetXtreme BCM5782 1000Base-T
@@ -10987,6 +11157,7 @@
 		0e11 00cb  NC7781 Gigabit Server Adapter (PCI-X, 10,100,1000-T)
 		103c 12c3  Combo FC/GigE-SX [A9782A]
 		103c 12ca  Combo FC/GigE-T [A9784A]
+		103c 1321  Core I/O LAN/SCSI Combo [AB314A]
 		14e4 0009  NetXtreme BCM5703 1000Base-T
 		14e4 000a  NetXtreme BCM5703 1000Base-SX
 	16dd  NetLink BCM5781 Gigabit Ethernet PCI Express
@@ -11346,7 +11517,7 @@
 	2f20  HSF 56k Data/Fax Modem
 		14f1 200c  Soft Data Fax Modem with SmartCP
 		14f1 200f  Dimension 3000
-	2f30  HSF 56k Data/Fax Modem
+	2f30  SoftV92 SpeakerPhone SoftRing Modem with SmartSP
 		14f1 2014  Devolo MikroLink 56K Modem PCI
 	2f50  Conexant SoftK56 Data/Fax Modem
 	5045  CX20549 (Venice)
@@ -11437,7 +11608,9 @@
 		18ac d820  DViCO FusionHDTV3 Gold-T
 		18ac db00  DVICO FusionHDTV DVB-T1
 	8852  CX23885 PCI Video and Audio Decoder
+		0070 8010  Hauppauge WinTV HVR-1400 ExpressCard
 		1461 c039  AVerTV Hybrid Express (A577)
+		18ac db78  FusionHDTV DVB-T Dual Express
 14f2  MOBILITY Electronics
 	0120  EV1000 bridge
 	0121  EV1000 Parallel port
@@ -11761,14 +11934,15 @@
 15aa  Moreton Bay
 15ab  Bluesteel Networks Inc
 15ac  North Atlantic Instruments
-15ad  VMware Inc
-	0405  Abstract SVGA II Adapter
-	0710  Abstract SVGA Adapter
-	0720  Abstract Ethernet Controller
+15ad  VMware
+	0405  SVGA II Adapter
+	0710  SVGA Adapter
+	0720  VMXNET Ethernet Controller
 	0740  Virtual Machine Communication Interface
-	0770  Abstract USB2 EHCI Controller
+	0770  USB2 EHCI Controller
 	0790  PCI bridge
-	07b0  VMXNET3 Ethernet Adapter
+	07a0  PCI Express Root Port
+	07b0  VMXNET3 Ethernet Controller
 	07c0  PVSCSI SCSI Controller
 	0801  Virtual Machine Interface
 		15ad 0800  Hypervisor ROM Interface
@@ -11797,6 +11971,22 @@
 15b4  CCI/TRIAD
 15b5  Cimetrics Inc
 15b6  Texas Memory Systems Inc
+	0001  XP15 DSP Accelerator
+	0002  XP30 DSP Accelerator
+	0003  XP00 Data Acquisition Device
+	0004  XP35 DSP Accelerator
+	0007  XP100 DSP Accelerator [XP100-T0]
+	0008  XP100 DSP Accelerator [XP100-T1]
+	0009  XP100 DSP Accelerator [XP100-E0]
+	000a  XP100 DSP Accelerator [XP100-E1]
+	000e  XP100 DSP Accelerator [XP100-0]
+	000f  XP100 DSP Accelerator [XP100-1]
+	0010  XP100 DSP Accelerator [XP100-P0]
+	0011  XP100 DSP Accelerator [XP100-P1]
+	0012  XP100 DSP Accelerator [XP100-P2]
+	0013  XP100 DSP Accelerator [XP100-P3]
+	0014  RamSan-20 Flash SSD
+	0015  ZBox
 15b7  Sandisk Corp
 15b8  ADDI-DATA GmbH
 	1003  APCI1032 SP controller (32 digi inputs w/ opto coupler)
@@ -11865,7 +12055,7 @@
 15e9  Pacific Digital Corp
 	1841  ADMA-100 DiscStaQ ATA Controller
 15ea  Tokyo Denshi Sekei K.K.
-15eb  Drsearch GmbH
+15eb  DResearch Digital Media Systems GmbH
 15ec  Beckhoff GmbH
 	3101  FC3101 Profibus DP 1 Channel PCI
 	5102  FC5102
@@ -12242,8 +12432,10 @@
 179c  Data Patterns
 	0557  DP-PCI-557 [PCI 1553B]
 	0566  DP-PCI-566 [Intelligent PCI 1553B]
-	1152  DP-cPCI-1152 (8channel Isolated ADC Module)
+	1152  DP-cPCI-1152 (8-channel Isolated ADC Module)
 	5031  DP-CPCI-5031-Synchro Module
+# cPCI Carrier for Mezzanine Modules
+	5112  DP-cPCI-5112 [MM-Carrier]
 	5121  DP-CPCI-5121-IP Carrier
 	5211  DP-CPCI-5211-IP Carrier
 	5679  AGE Display Module
@@ -12363,6 +12555,9 @@
 	0681  RT2860 Wireless 802.11n PCIe
 	0701  RT2760 Wireless 802.11n 1T/2R Cardbus
 	0781  RT2860
+	3090  RT3090 Wireless 802.11n 1T/1R PCIe
+	3091  RT3091 Wireless 802.11n 1T/2R PCIe
+	3092  RT3092 Wireless 802.11n 2T/2R PCIe
 	e932  RT2560F 802.11 b/g PCI
 1815  Devolo AG
 1820  InfiniCon Systems Inc.
@@ -12452,6 +12647,8 @@
 18c9  ARVOO Engineering BV
 18ca  XGI Technology Inc. (eXtreme Graphics Innovation)
 	0020  Volari Z7/Z9/Z9s
+	0021  Volari Z9M
+	0027  Volari Z11M
 	0040  Volari V3XT/V5/V8
 	0047  Volari 8300 (chip: XP10, codename: XG47)
 18d2  Sitecom
@@ -12538,6 +12735,9 @@
 		1924 5102  SFN4111T-A2
 		1924 5103  SFN4111T-R3
 		1924 5104  SFN4111T-R4
+		1924 5105  SFN4111T-R5
+		1924 5201  SFN4112F-R1
+		1924 5202  SFN4112F-R2
 	6703  SFC4000 rev A iSCSI/Onload [Solarstorm]
 		10b8 0102  SMC10GPCIe-10BT (A2) [TigerCard]
 		10b8 0103  SMC10GPCIe-10BT (A3) [TigerCard]
@@ -12623,7 +12823,7 @@
 1966  Orad Hi-Tec Systems
 	1975  DVG64 family
 1969  Attansic Technology Corp.
-	1026  L1e Gigabit Ethernet Adapter
+	1026  Atheros AR8121/AR8113/AR8114 PCI-E Ethernet Controller
 		1043 8304  P5KPL-CM Motherboard
 	1048  L1 Gigabit Ethernet Adapter
 		1043 8226  P5KPL-VM Motherboard
@@ -12646,9 +12846,10 @@
 	2360  JMicron 20360/20363 AHCI Controller
 	2361  JMB361 AHCI/IDE
 		1462 7235  P965 Neo MS-7235 mainboard
-	2363  JMicron 20360/20363 AHCI Controller
+	2363  20360/20363 Serial ATA Controller
 		1043 81e4  P5B [JMB363]
 		1458 b000  GA-EP45-DS5 Motherboard
+		1849 2363  Motherboard (one of many)
 	2365  JMB365 AHCI/IDE
 	2366  JMB366 AHCI/IDE
 	2368  JMB368 IDE controller
@@ -12737,7 +12938,7 @@
 1a77  Lightfleet Corporation
 1a78  Virident Systems Inc.
 1a84  Commex Technologies
-	0001  Thunder 10 Gigabit ethernet
+	0001  Vulcan SP HT6210 10 Gigabit Ethernet (rev 02)
 1a8c  Verigy Pte. Ltd.
 	1100  E8001-66443 PCI Express CIC
 1a8e  DRS Technologies
@@ -12762,12 +12963,14 @@
 1af4  Qumranet, Inc.
 	1000  Virtio network device
 	1001  Virtio block device
+	1002  Virtio memory balloon
+	1003  Virtio console
 1af5  Netezza Corp.
 1afa  J & W Electronics Co., Ltd.
 1b13  Jaton Corp
+1b1a  K&F Computing Research Co.
+	0e70  GRAPE
 1b36  Red Hat, Inc.
-	1000  Virtio network device
-	1001  Virtio block device
 1b3e  Teradata Corp.
 	1fa8  BYNET BIC2SE/X
 		1b3e 00a3  BYNET BIC2SX
@@ -12852,7 +13055,6 @@
 3475  Arastra Inc.
 3513  ARCOM Control Systems Ltd
 3842  eVga.com. Corp.
-	c370  e-GeFORCE 6600 256 DDR PCI-e
 38ef  4Links
 3d3d  3DLabs
 	0001  GLINT 300SX
@@ -12936,8 +13138,9 @@
 		103c 3251  NC375i 1G w/NC524SFP 10G Module
 		103c 705a  NC375i Integrated Quad Port Multifunction Gigabit Server Adapter
 		103c 705b  NC522SFP Dual Port 10GbE Server Adapter
-		4040 0124  NX3-4GBT Low-profile PCIE 2.0 card with 4 Gigabit Ethernet ports
-		4040 0126  NX3-20GxR 2-port 10GbE low-profile PCIE 2.0 card with pluggable SFP+ interconnect modules
+		152d 896b  TG20 Dual Port 10GbE Server/Storage Adapter
+		4040 0124  NX3031 Quad Port Gigabit Server Adapter
+		4040 0126  Dual Port SFP+ 10GbE Server Adapter
 4143  Digital Equipment Corp
 4144  Alpha Data
 	0044  ADM-XRCIIPro
@@ -12973,7 +13176,39 @@
 		10fc d039  GV-MVP/RX2W (2nd unit)
 		12ab fff3  MPG600
 		12ab ffff  MPG600
-		1461 c019  UltraTV 1500 MCE
+		1461 c00a  M113 PCI Analog TV (PAL/SECAM, Philips FQ1216MK3 tuner)
+		1461 c00b  M113 PCI Analog TV (PAL/SECAM+FM, Philips FM1216MK3 tuner)
+		1461 c00c  M113 PCI Analog TV (NTSC, JAPAN version, Philips FI1286MK2 tuner)
+		1461 c010  M113 PCI Analog TV (NTSC, Philips FI1236MK3 tuner)
+		1461 c011  M113 PCI Analog TV (NTSC+FM, Philips FM1236MK3 tuner)
+		1461 c018  M113 PCI Analog TV (NTSC, Philips FQ1236MK5 tuner)
+		1461 c019  UltraTV 1500 MCE, a.k.a. M113 PCI Analog TV (NTSC+FM, Philips FQ1236MK5 tuner)
+		1461 c01a  M113 PCI Analog TV (PAL/SECAM, Philips FQ1216MK5 tuner)
+		1461 c01b  M113 PCI Analog TV (PAL/SECAM+FM, Philips FM1216MK5 tuner)
+		1461 c030  M113 PCI Analog TV (NTSC-J, Partsnic tuner)
+		1461 c031  M113 PCI Analog TV (NTSC-J+FM, Partsnic tuner)
+		1461 c032  M113 PCI Analog TV (PAL/SECAM, Partsnic tuner)
+		1461 c033  M113 PCI Analog TV (PAL/SECAM+FM, Partsnic tuner)
+		1461 c034  M113 PCI Analog TV (NTSC, Partsnic tuner)
+		1461 c035  M113 PCI Analog TV (NTSC+FM, Partsnic tuner)
+		1461 c03f  C115 PCI video capture card (no tuner)
+		1461 c136  M104 mini-PCI Analog TV
+		1461 c20a  M755 AVerTV Video Capture (PAL/SECAM, Philips FQ1216MK3 tuner)
+		1461 c218  M755 AVerTV Video Capture (NTSC, Philips FQ1236MK5 tuner)
+		1461 c219  M755 AVerTV Video Capture (NTSC+FM, Philips FQ1236MK5 tuner)
+		1461 c21a  M755 AVerTV Video Capture (PAL/SECAM, Philips FQ1216MK5 tuner)
+		1461 c21b  M755 AVerTV Video Capture (PAL/SECAM+FM, Philips FM1216MK5 tuner)
+		1461 c230  M755 AVerTV Video Capture (NTSC-J, Partsnic tuner)
+		1461 c231  M755 AVerTV Video Capture (NTSC-J+FM, Partsnic tuner)
+		1461 c232  M755 AVerTV Video Capture (PAL/SECAM, Partsnic tuner)
+		1461 c233  M755 AVerTV Video Capture (PAL/SECAM+FM, Partsnic tuner)
+		1461 c234  M755 AVerTV Video Capture (NTSC, Partsnic tuner)
+		1461 c235  M755 AVerTV Video Capture (NTSC+FM, Partsnic tuner)
+		1461 c337  E106 AVerMedia AVerTV Video Capture
+		1461 c439  M116 AVerMedia AVerTV MCE 116 Plus (NTSC/PAL/SECAM+FM+REMOTE, Xceive 2028 tuner)
+		1461 c5ff  C755 AVerTV Video Capture card (no tuner)
+		1461 c6ff  C115 PCI video capture card (no tuner)
+		1461 c739  M785 AVerMedia PCI Analog TV (NTSC/PAL/SECAM+FM, Xceive 2028 tuner)
 		9005 0092  VideOh! AVC-2010
 		9005 0093  VideOh! AVC-2410
 	0803  iTVC15 MPEG-2 Encoder
@@ -13150,6 +13385,7 @@
 		5333 9143  86C394-397 Savage4 SGRAM 143
 	8a23  Savage 4
 	8a25  ProSavage PM133
+		0303 0303  D9840-60001 [Brio BA410 Motherboard]
 	8a26  ProSavage KM133
 	8c00  ViRGE/M3
 	8c01  ViRGE/MX
@@ -13375,7 +13611,7 @@
 	100f  82545EM Gigabit Ethernet Controller (Copper)
 		1014 0269  iSeries 1000/100/10 Ethernet Adapter
 		1014 028e  PRO/1000 MT Network Connection
-		15ad 0750  Abstract PRO/1000 MT Single Port Adapter
+		15ad 0750  PRO/1000 MT Single Port Adapter
 		8086 1000  PRO/1000 MT Network Connection
 		8086 1001  PRO/1000 MT Server Adapter
 	1010  82546EB Gigabit Ethernet Controller (Copper)
@@ -13683,8 +13919,10 @@
 		8086 a12c  10 Gigabit AT Server Adapter
 	10c9  82576 Gigabit Network Connection
 		103c 323f  NC362i Integrated Dual port Gigabit Server Adapter
+		10a9 8028  UV-BaseIO dual-port GbE
 		8086 a01c  Gigabit ET Dual Port Server Adapter
 		8086 a03c  Gigabit ET Dual Port Server Adapter
+	10ca  82576 Virtual Function
 	10cb  82567V Gigabit Network Connection
 	10cc  82567LM-2 Gigabit Network Connection
 	10cd  82567LF-2 Gigabit Network Connection
@@ -13705,7 +13943,7 @@
 	10dd  82598EB 10 Gigabit AT CX4 Network Connection
 	10de  82567LM-3 Gigabit Network Connection
 	10df  82567LF-3 Gigabit Network Connection
-	10e1  82598EB 10-Gigabit AT Dual Port Network Connection
+	10e1  82598EB 10 Gigabit AF Dual Port Network Connection
 		8086 a15f  10-Gigabit SR Dual Port Express Module
 	10e2  82575GB Gigabit Network Connection
 		8086 10e2  Gigabit VT Quad Port Server Adapter
@@ -13715,16 +13953,20 @@
 		8086 a02f  Gigabit EF Dual Port Server Adapter
 	10e7  82576 Gigabit Network Connection
 		103c 31ff  NC362i Integrated Dual Port BL-c Gigabit Server Adapter
+	10e8  82576 Gigabit Network Connection
+		8086 a02c  Gigabit ET Quad Port Server Adapter
 	10ec  82598EB 10-Gigabit AT CX4 Network Connection
 		8086 a01f  10 Gigabit CX4 Dual Port Server Adapter
 		8086 a11f  10 Gigabit CX4 Dual Port Server Adapter
-	10f1  82598EB 10-Gigabit AT Dual Port Network Connection
+	10f1  82598EB 10-Gigabit AF Dual Port Network Connection
 		8086 a20f  10-Gigabit AF DA Dual Port Server Adapter
 		8086 a21f  10-Gigabit AF DA Dual Port Server Adapter
-	10f4  82598EB 10-Gigabit AT Network Connection
+	10f4  82598EB 10-Gigabit AF Network Connection
 		8086 106f  10 Gigabit XF LR Server Adapter
 		8086 a06f  10-Gigabit XF LR Server Adapter
 	10f5  82567LM Gigabit Network Connection
+	10f6  82574L Gigabit Network Connection
+	10fe  82552 10/100 Network Connection
 	1107  PRO/1000 MF Server Adapter (LX)
 	1130  82815 815 Chipset Host Bridge and Memory Controller Hub
 		1025 1016  Travelmate 612 TX
@@ -13924,6 +14166,7 @@
 	1234  430MX - 82371MX Mobile PCI I/O IDE Xcelerator (MPIIX)
 	1235  430MX - 82437MX Mob. System Ctrlr (MTSC) & 82438MX Data Path (MTDP)
 	1237  440FX - 82441FX PMC [Natoma]
+		1af4 1100  Qemu virtual machine
 	1239  82371FB PIIX IDE Interface
 	123b  82380PB PCI to PCI Docking Bridge
 	123c  82380AB (MISA) Mobile PCI-to-ISA Bridge
@@ -13943,6 +14186,8 @@
 		4c53 1090  Cx9/Vx9 mainboard
 	1462  82870P2 P64H2 Hot Plug Controller
 	1508  82598EB Gigabit BX Network Connection
+	150a  82576NS Gigabit Ethernet Controller
+	150c  82583V Gigabit Network Connection
 	1960  80960RP (i960RP) Microprocessor
 		101e 0431  MegaRAID 431 RAID Controller
 		101e 0438  MegaRAID 438 Ultra2 LVD RAID Controller
@@ -14167,6 +14412,7 @@
 		1014 0503  ThinkPad R31
 		1014 051a  ThinkPad A/T/X Series
 		101f 1025  620 Series
+		1043 1496  PCtel HSP56 MR
 		104d 80e7  VAIO PCG-GR214EP/GR214MP/GR215MP/GR314MP/GR315MP
 		134d 4c21  Dell Inspiron 2100 internal modem
 		144d 2115  vpr Matrix 170B4 internal modem
@@ -14395,7 +14641,7 @@
 		1014 02dd  eServer xSeries server mainboard
 		1014 02ed  eServer xSeries server mainboard
 		1028 0169  Precision 470
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 onboard UHCI
 		1028 016d  PowerEdge 2850 onboard UHCI
 		1028 0170  PowerEdge 6850 onboard UHCI
 		1028 0183  PowerEdge 1800
@@ -14432,7 +14678,7 @@
 		1014 02dd  eServer xSeries server mainboard
 		1014 02ed  eServer xSeries server mainboard
 		1028 0169  Precision 470
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 onboard UHCI
 		1028 016d  PowerEdge 2850 onboard UHCI
 		1028 0170  PowerEdge 6850 onboard UHCI
 		1028 0183  PowerEdge 1800
@@ -14468,7 +14714,7 @@
 	24d7  82801EB/ER (ICH5/ICH5R) USB UHCI Controller #3
 		1014 02ed  xSeries server mainboard
 		1028 0169  Precision 470
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 onboard UHCI
 		1028 016d  PowerEdge 2850 onboard UHCI
 		1028 0170  PowerEdge 6850 onboard UHCI
 		1028 0183  PowerEdge 1800
@@ -14488,7 +14734,7 @@
 		1014 02dd  eServer xSeries server mainboard
 		1014 02ed  eServer xSeries server mainboard
 		1028 0169  Precision 470
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 IDE Controller
 		1028 016d  PowerEdge 2850 IDE Controller
 		1028 0170  PowerEdge 6850 IDE Controller
 		1028 019a  PowerEdge SC1425
@@ -14511,7 +14757,7 @@
 		1014 02dd  eServer xSeries server mainboard
 		1014 02ed  eServer xSeries server mainboard
 		1028 0169  Precision 470
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 onboard EHCI
 		1028 016d  PowerEdge 2850 onboard EHCI
 		1028 0170  PowerEdge 6850 onboard EHCI
 		1028 0183  PowerEdge 1800
@@ -15224,6 +15470,7 @@
 		1462 7235  P965 Neo MS-7235 mainboard
 	2828  82801HBM/HEM (ICH8M/ICH8M-E) SATA IDE Controller
 		1028 01f3  Inspiron 1420
+		103c 30c0  Compaq 6710b
 		e4bf cc47  CCG-RUMBA
 	2829  82801HBM/HEM (ICH8M/ICH8M-E) SATA AHCI Controller
 		1025 0121  Aspire 5920G
@@ -15380,6 +15627,9 @@
 		1028 0211  Optiplex 755
 		1028 023c  PowerEdge R200 onboard SATA Controller
 	2921  82801IB (ICH9) 2 port SATA IDE Controller
+		1028 0235  PowerEdge R710 SATA IDE Controller
+		1028 0236  PowerEdge R610 SATA IDE Controller
+		1028 0237  PowerEdge T610 SATA IDE Controller
 	2922  82801IR/IO/IH (ICH9R/DO/DH) 6 port SATA AHCI Controller
 	2923  82801IB (ICH9) 4 port SATA AHCI Controller
 	2925  82801IR/IO (ICH9R/DO) SATA RAID Controller
@@ -15401,38 +15651,69 @@
 		1028 020f  PowerEdge R300 onboard UHCI
 		1028 0210  PowerEdge T300 onboard UHCI
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB UHCI Controller
+		1028 0236  PowerEdge R610 USB UHCI Controller
+		1028 0237  PowerEdge T610 USB UHCI Controller
 		1028 023c  PowerEdge R200 onboard UHCI
+		1028 0287  PowerEdge M610 onboard UHCI
+		1028 029c  PowerEdge M710 USB UHCI Controller
 		1028 2011  Optiplex 755
 	2935  82801I (ICH9 Family) USB UHCI Controller #2
 		1028 020f  PowerEdge R300 onboard UHCI
 		1028 0210  PowerEdge T300 onboard UHCI
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB UHCI Controller
+		1028 0236  PowerEdge R610 USB UHCI Controller
+		1028 0237  PowerEdge T610 USB UHCI Controller
 		1028 023c  PowerEdge R200 onboard UHCI
+		1028 0287  PowerEdge M610 onboard UHCI
+		1028 029c  PowerEdge M710 USB UHCI Controller
 	2936  82801I (ICH9 Family) USB UHCI Controller #3
 		1028 020f  PowerEdge R300 onboard UHCI
 		1028 0210  PowerEdge T300 onboard UHCI
 		1028 0211  Optiplex 755
+		1028 0237  PowerEdge T610 USB UHCI Controller
 		1028 023c  PowerEdge R200 onboard UHCI
+		1028 0287  PowerEdge M610 onboard UHCI
+		1028 029c  PowerEdge M710 USB UHCI Controller
 	2937  82801I (ICH9 Family) USB UHCI Controller #4
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB UHCI Controller
+		1028 0236  PowerEdge R610 USB UHCI Controller
+		1028 0237  PowerEdge T610 USB UHCI Controller
 		1028 0287  PowerEdge M610 onboard UHCI
+		1028 029c  PowerEdge M710 USB UHCI Controller
 		1028 2011  Optiplex 755
 		8086 2937  Optiplex 755
 		8086 2942  828011 (ICH9 Family ) USB UHCI Controller
 	2938  82801I (ICH9 Family) USB UHCI Controller #5
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB UHCI Controller
+		1028 0236  PowerEdge R610 USB UHCI Controller
+		1028 0237  PowerEdge T610 USB UHCI Controller
 		1028 0287  PowerEdge M610 onboard UHCI
+		1028 029c  PowerEdge M710 USB UHCI Controller
 		8086 2938  Optiplex 755
 	2939  82801I (ICH9 Family) USB UHCI Controller #6
 		1028 0210  PowerEdge T300 onboard UHCI
+		1028 0237  PowerEdge T610 USB UHCI Controller
 	293a  82801I (ICH9 Family) USB2 EHCI Controller #1
 		1028 020f  PowerEdge R300 onboard EHCI
 		1028 0210  PowerEdge T300 onboard EHCI
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB EHCI Controller
+		1028 0236  PowerEdge R610 USB EHCI Controller
+		1028 0237  PowerEdge T610 USB EHCI Controller
 		1028 023c  PowerEdge R200 onboard EHCI
+		1028 0287  PowerEdge M610 onboard EHCI
+		1028 029c  PowerEdge M710 USB EHCI Controller
 	293c  82801I (ICH9 Family) USB2 EHCI Controller #2
 		1028 0211  Optiplex 755
+		1028 0235  PowerEdge R710 USB EHCI Controller
+		1028 0236  PowerEdge R610 USB EHCI Controller
+		1028 0237  PowerEdge T610 USB EHCI Controller
 		1028 0287  PowerEdge M610 onboard EHCI
+		1028 029c  PowerEdge M710 USB EHCI Controller
 		8086 293c  Optiplex 755
 	293e  82801I (ICH9 Family) HD Audio Controller
 		1028 0211  Optiplex 755
@@ -15620,34 +15901,34 @@
 	2b5c  Nehalem-EX QPI Physical Port 3: REUT control/status
 	2b5e  Nehalem-EX QPI Physical Port 3: Misc. control/status
 	2b60  Nehalem-EX SMI Physical Port 0: REUT control/status
-	2b62  Nehalem-EX SMI Physical Port 0: REUT control/status
-	2b64  Nehalem-EX SMI Physical Port 1: Misc control/status
+	2b62  Nehalem-EX SMI Physical Port 0: Misc control/status
+	2b64  Nehalem-EX SMI Physical Port 0: REUT control/status
 	2b66  Nehalem-EX SMI Physical Port 1: Misc control/status
 	2b68  Nehalem-EX System Configuration Controller 3
 	2b6a  Nehalem-EX System Configuration Controller 4
-	2c01  Core i7 QuickPath Architecture System Address Decoder
-	2c10  Core i7 QPI Link 0
-	2c11  Core i7 QPI Physical 0
-	2c14  Core i7 QPI Link 1
-	2c15  Core i7 QPI Physical 1
-	2c18  Core i7 Integrated Memory Controller
-	2c19  Core i7 Integrated Memory Controller Target Address Decoder
-	2c1a  Core i7 Integrated Memory Controller RAS Registers
-	2c1c  Core i7 Integrated Memory Controller Test Registers
-	2c20  Core i7 Integrated Memory Controller Channel 0 Control Registers
-	2c21  Core i7 Integrated Memory Controller Channel 0 Address Registers
-	2c22  Core i7 Integrated Memory Controller Channel 0 Rank Registers
-	2c23  Core i7 Integrated Memory Controller Channel 0 Thermal Control Registers
-	2c28  Core i7 Integrated Memory Controller Channel 1 Control Registers
-	2c29  Core i7 Integrated Memory Controller Channel 1 Address Registers
-	2c2a  Core i7 Integrated Memory Controller Channel 1 Rank Registers
-	2c2b  Core i7 Integrated Memory Controller Channel 1 Thermal Control Registers
-	2c30  Core i7 Integrated Memory Controller Channel 2 Control Registers
-	2c31  Core i7 Integrated Memory Controller Channel 2 Address Registers
-	2c32  Core i7 Integrated Memory Controller Channel 2 Rank Registers
-	2c33  Core i7 Integrated Memory Controller Channel 2 Thermal Control Registers
-	2c40  Core i7 QuickPath Architecture Generic Non-Core Registers
-	2c41  QuickPath Architecture Generic Non-Core Registers
+	2c01  Xeon 5500/Core i7 QuickPath Architecture System Address Decoder
+	2c10  Xeon 5500/Core i7 QPI Link 0
+	2c11  Xeon 5500/Core i7 QPI Physical 0
+	2c14  Xeon 5500/Core i7 QPI Link 1
+	2c15  Xeon 5500/Core i7 QPI Physical 1
+	2c18  Xeon 5500/Core i7 Integrated Memory Controller
+	2c19  Xeon 5500/Core i7 Integrated Memory Controller Target Address Decoder
+	2c1a  Xeon 5500/Core i7 Integrated Memory Controller RAS Registers
+	2c1c  Xeon 5500/Core i7 Integrated Memory Controller Test Registers
+	2c20  Xeon 5500/Core i7 Integrated Memory Controller Channel 0 Control Registers
+	2c21  Xeon 5500/Core i7 Integrated Memory Controller Channel 0 Address Registers
+	2c22  Xeon 5500/Core i7 Integrated Memory Controller Channel 0 Rank Registers
+	2c23  Xeon 5500/Core i7 Integrated Memory Controller Channel 0 Thermal Control Registers
+	2c28  Xeon 5500/Core i7 Integrated Memory Controller Channel 1 Control Registers
+	2c29  Xeon 5500/Core i7 Integrated Memory Controller Channel 1 Address Registers
+	2c2a  Xeon 5500/Core i7 Integrated Memory Controller Channel 1 Rank Registers
+	2c2b  Xeon 5500/Core i7 Integrated Memory Controller Channel 1 Thermal Control Registers
+	2c30  Xeon 5500/Core i7 Integrated Memory Controller Channel 2 Control Registers
+	2c31  Xeon 5500/Core i7 Integrated Memory Controller Channel 2 Address Registers
+	2c32  Xeon 5500/Core i7 Integrated Memory Controller Channel 2 Rank Registers
+	2c33  Xeon 5500/Core i7 Integrated Memory Controller Channel 2 Thermal Control Registers
+	2c40  Xeon 5500/Core i7 QuickPath Architecture Generic Non-Core Registers
+	2c41  Xeon 5500/Core i7 QuickPath Architecture Generic Non-Core Registers
 	2c50  QuickPath Architecture Generic Non-Core Registers
 	2c51  QuickPath Architecture Generic Non-Core Registers
 	2c52  QuickPath Architecture Generic Non-Core Registers
@@ -15656,6 +15937,14 @@
 	2c55  QuickPath Architecture Generic Non-Core Registers
 	2c56  QuickPath Architecture Generic Non-Core Registers
 	2c57  QuickPath Architecture Generic Non-Core Registers
+	2c58  Jasper Forest QPI Generic Non-core Registers
+	2c59  Jasper Forest QPI Generic Non-core Registers
+	2c5a  Jasper Forest QPI Generic Non-core Registers
+	2c5b  Jasper Forest QPI Generic Non-core Registers
+	2c5c  Jasper Forest QPI Generic Non-core Registers
+	2c5d  Jasper Forest QPI Generic Non-core Registers
+	2c5e  Jasper Forest QPI Generic Non-core Registers
+	2c5f  Jasper Forest QPI Generic Non-core Registers
 	2c81  QuickPath Architecture System Address Decoder
 	2c90  QPI Link
 	2c91  QPI Physical 0
@@ -15670,6 +15959,27 @@
 	2ca9  Lynnfield Integrated Memory Controller Channel 1 Address Registers
 	2caa  Lynnfield Integrated Memory Controller Channel 1 Rank Registers
 	2cab  Lynnfield Integrated Memory Controller Channel 1 Thermal Control Registers
+	2cc1  Jasper Forest QPI System Address Decoder
+	2cd0  Jasper Forest QPI Link 0
+	2cd1  Jasper Forest QPI Physical 0
+	2cd4  Jasper Forest QPI Link 1
+	2cd5  Jasper Forest QPI Physical 1
+	2cd8  Jasper Forest Integrated Memory Controller Registers
+	2cd9  Jasper Forest Integrated Memory Controller Target Address Decoder
+	2cda  Jasper Forest Integrated Memory Controller RAS Registers
+	2cdc  Jasper Forest Integrated Memory Controller Test Registers
+	2ce0  Jasper Forest Integrated Memory Controller Channel 0 Control
+	2ce1  Jasper Forest Integrated Memory Controller Channel 0 Address
+	2ce2  Jasper Forest Integrated Memory Controller Channel 0 Rank
+	2ce3  Jasper Forest Integrated Memory Controller Channel 0 Thermal Control
+	2ce8  Jasper Forest Integrated Memory Controller Channel 1 Control
+	2ce9  Jasper Forest Integrated Memory Controller Channel 1 Address
+	2cea  Jasper Forest Integrated Memory Controller Channel 1 Rank
+	2ceb  Jasper Forest Integrated Memory Controller Channel 1 Thermal Control
+	2cf0  Jasper Forest Integrated Memory Controller Channel 2 Control
+	2cf1  Jasper Forest Integrated Memory Controller Channel 2 Address
+	2cf2  Jasper Forest Integrated Memory Controller Channel 2 Rank
+	2cf3  Jasper Forest Integrated Memory Controller Channel 2 Thermal Control
 	2e00  4 Series Chipset DRAM Controller
 	2e01  4 Series Chipset PCI Express Root Port
 	2e02  4 Series Chipset Integrated Graphics Controller
@@ -15739,47 +16049,47 @@
 	3382  81342 [Chevelon] I/O Processor (ATUe)
 	33c3  IOP348 I/O Processor (SL8De) in IOC Mode SAS/SATA
 	33cb  IOP348 I/O Processor (SL8Dx) in IOC Mode SAS/SATA
-	3400  X58 I/O Hub to ESI Port
-	3401  X58 I/O Hub to ESI Port
-	3402  X58 I/O Hub to ESI Port
-	3403  X58 I/O Hub to ESI Port
+	3400  5520/5500/X58 I/O Hub to ESI Port
+	3401  5520/5500/X58 I/O Hub to ESI Port
+	3402  5520/5500/X58 I/O Hub to ESI Port
+	3403  5520/5500/X58 I/O Hub to ESI Port
 		1028 0287  PowerEdge M610 I/O Hub to ESI Port
-	3404  X58 I/O Hub to ESI Port
-	3405  X58 I/O Hub to ESI Port
-	3406  X58 I/O Hub to ESI Port
-	3407  X58 I/O Hub to ESI Port
-	3408  X58 I/O Hub PCI Express Root Port 1
-	3409  X58 I/O Hub PCI Express Root Port 2
-	340a  X58 I/O Hub PCI Express Root Port 3
-	340b  X58 I/O Hub PCI Express Root Port 4
-	340c  X58 I/O Hub PCI Express Root Port 5
-	340d  X58 I/O Hub PCI Express Root Port 6
-	340e  X58 I/O Hub PCI Express Root Port 7
-	340f  X58 I/O Hub PCI Express Root Port 8
-	3410  X58 I/O Hub PCI Express Root Port 9
-	3411  X58 I/O Hub PCI Express Root Port 10
-	3418  X58 Physical Layer Port 0
-	3419  Quickpath Interconnect Physical Layer Port 1
-	3420  X58 I/O Hub PCI Express Root Port 0
-	3421  X58 I/O Hub PCI Express Root Port 0
-	3422  X58 I/O Hub GPIO and Scratch Pad Registers
-	3423  X58 I/O Hub Control Status and RAS Registers
-	3425  X58 Physical and Link Layer Registers Port 0
-	3426  X58 Routing and Protocol Layer Registers Port 0
-	3427  QuickPath Interconnect Physical and Link Layer Registers Port 1
-	3428  QuickPath Interconnect Routing and Protocol Layer Registers Port 1
-	3429  X58 Chipset QuickData Technology Device
-	342a  X58 Chipset QuickData Technology Device
-	342b  X58 Chipset QuickData Technology Device
-	342c  DMA Engine
-	342d  X58 I/O Hub I/OxAPIC Interrupt Controller
-	342e  X58 I/O Hub System Management Registers
-	342f  X58 Trusted Execution Technology Registers
-	3430  X58 Chipset QuickData Technology Device
-	3431  X58 Chipset QuickData Technology Device
-	3432  X58 Chipset QuickData Technology Device
-	3433  X58 Chipset QuickData Technology Device
-	3438  X58 I/O Hub Throttle Registers
+	3404  5520/5500/X58 I/O Hub to ESI Port
+	3405  5520/5500/X58 I/O Hub to ESI Port
+	3406  5520/5500/X58 I/O Hub to ESI Port
+	3407  5520/5500/X58 I/O Hub to ESI Port
+	3408  5520/5500/X58 I/O Hub PCI Express Root Port 1
+	3409  5520/5500/X58 I/O Hub PCI Express Root Port 2
+	340a  5520/5500/X58 I/O Hub PCI Express Root Port 3
+	340b  5520/X58 I/O Hub PCI Express Root Port 4
+	340c  5520/X58 I/O Hub PCI Express Root Port 5
+	340d  5520/X58 I/O Hub PCI Express Root Port 6
+	340e  5520/5500/X58 I/O Hub PCI Express Root Port 7
+	340f  5520/5500/X58 I/O Hub PCI Express Root Port 8
+	3410  5520/5500/X58 I/O Hub PCI Express Root Port 9
+	3411  5520/5500/X58 I/O Hub PCI Express Root Port 10
+	3418  5520/5500/X58 Physical Layer Port 0
+	3419  5520/5500 Physical Layer Port 1
+	3420  5520/5500/X58 I/O Hub PCI Express Root Port 0
+	3421  5520/5500/X58 I/O Hub PCI Express Root Port 0
+	3422  5520/5500/X58 I/O Hub GPIO and Scratch Pad Registers
+	3423  5520/5500/X58 I/O Hub Control Status and RAS Registers
+	3425  5520/5500/X58 Physical and Link Layer Registers Port 0
+	3426  5520/5500/X58 Routing and Protocol Layer Registers Port 0
+	3427  5520/5500 Physical and Link Layer Registers Port 1
+	3428  5520/5500 Routing & Protocol Layer Register Port 1
+	3429  5520/5500/X58 Chipset QuickData Technology Device
+	342a  5520/5500/X58 Chipset QuickData Technology Device
+	342b  5520/5500/X58 Chipset QuickData Technology Device
+	342c  5520/5500/X58 Chipset QuickData Technology Device
+	342d  5520/5500/X58 I/O Hub I/OxAPIC Interrupt Controller
+	342e  5520/5500/X58 I/O Hub System Management Registers
+	342f  5520/5500/X58 Trusted Execution Technology Registers
+	3430  5520/5500/X58 Chipset QuickData Technology Device
+	3431  5520/5500/X58 Chipset QuickData Technology Device
+	3432  5520/5500/X58 Chipset QuickData Technology Device
+	3433  5520/5500/X58 Chipset QuickData Technology Device
+	3438  5520/5500/X58 I/O Hub Throttle Registers
 	3500  6311ESB/6321ESB PCI Express Upstream Port
 	3501  6310ESB PCI Express Upstream Port
 	3504  6311ESB/6321ESB I/OxAPIC Interrupt Controller
@@ -15860,7 +16170,7 @@
 	358e  82854 GMCH Integrated Graphics Device
 	3590  E7520 Memory Controller Hub
 		1014 02dd  eServer xSeries server mainboard
-		1028 016c  PowerEdge 1850
+		1028 016c  PowerEdge 1850 Memory Controller Hub
 		1028 016d  PowerEdge 2850 Memory Controller Hub
 		1028 019a  PowerEdge SC1425
 		1734 103e  PRIMERGY RX/TX S2 series
@@ -15910,6 +16220,48 @@
 	360e  7300 Chipset Debug and Miscellaneous Registers
 	360f  7300 Chipset FBD Branch 0 Registers
 	3610  7300 Chipset FBD Branch 1 Registers
+	3700  Jasper Forest DMI
+	3701  Jasper Forest DMI
+	3702  Jasper Forest DMI
+	3703  Jasper Forest DMI
+	3704  Jasper Forest DMI
+	3705  Jasper Forest DMI
+	3706  Jasper Forest DMI
+	3707  Jasper Forest DMI
+	3708  Jasper Forest DMI
+	3709  Jasper Forest DMI
+	370a  Jasper Forest DMI
+	370b  Jasper Forest DMI
+	370c  Jasper Forest DMI
+	370d  Jasper Forest DMI
+	370e  Jasper Forest DMI
+	370f  Jasper Forest DMI
+	3710  Jasper Forest CB3 DMA
+	3711  Jasper Forest CB3 DMA
+	3712  Jasper Forest CB3 DMA
+	3713  Jasper Forest CB3 DMA
+	3714  Jasper Forest CB3 DMA
+	3715  Jasper Forest CB3 DMA
+	3716  Jasper Forest CB3 DMA
+	3717  Jasper Forest CB3 DMA
+	3718  Jasper Forest CB3 DMA
+	3719  Jasper Forest CB3 DMA
+	371a  Jasper Forest QPI Link
+	371b  Jasper Forest QPI Routing and Protocol
+	371d  Jasper Forest QPI Routing and Protocol
+	3721  Jasper Forest PCI Express Root Port 1
+	3722  Jasper Forest PCI Express Root Port 2
+	3723  Jasper Forest PCI Express Root Port 3
+	3724  Jasper Forest PCI Express Root Port 4
+	3725  Jasper Forest NTB Primary
+	3726  Jasper Forest NTB Primary
+	3727  Jasper Forest NTB Secondary
+	3728  Jasper Forest Core
+	3729  Jasper Forest Core
+	372a  Jasper Forest Core
+	372b  Jasper Forest Core
+	372c  Jasper Forest Reserved
+	373f  Jasper Forest IOxAPIC
 	3a00  82801JD/DO (ICH10 Family) 4-port SATA IDE Controller
 	3a02  82801JD/DO (ICH10 Family) SATA AHCI Controller
 	3a05  82801JD/DO (ICH10 Family) SATA RAID Controller
@@ -16190,8 +16542,11 @@
 	65fa  5100 Chipset PCI Express x16 Port 4-7
 	65ff  5100 Chipset DMA Engine
 	7000  82371SB PIIX3 ISA [Natoma/Triton II]
+		1af4 1100  Qemu virtual machine
 	7010  82371SB PIIX3 IDE [Natoma/Triton II]
+		1af4 1100  Qemu virtual machine
 	7020  82371SB PIIX3 USB [Natoma/Triton II]
+		1af4 1100  Qemu virtual machine
 	7030  430VX - 82437VX TVX [Triton VX]
 	7050  Intercast Video Capture Card
 	7051  PB 642365-003 (Business Video Conferencing Card)
@@ -16204,6 +16559,7 @@
 		15ad 1976  Virtual Machine Chipset
 	7113  82371AB/EB/MB PIIX4 ACPI
 		15ad 1976  Virtual Machine Chipset
+		1af4 1100  Qemu virtual machine
 	7120  82810 GMCH (Graphics Memory Controller Hub)
 		4c53 1040  CL7 mainboard
 		4c53 1060  PC7 mainboard
@@ -16357,6 +16713,7 @@
 	7664  High Definition Audio Codec
 	7670  9770 High Definition Audio
 	7672  9772 High Definition Audio
+	7682  IDT High Definition Audio Codec
 	7690  9200 HD Audio Codec
 8401  TRENDware International Inc.
 8686  ScaleMP
@@ -16731,10 +17088,16 @@
 		1000 0002  2S (16C550 UART)
 		1000 0012  1P2S
 	9845  PCI 9845 Multi-I/O Controller
+# Serial ports at BAR0-3
 		1000 0004  0P4S (4 port 16550A serial card)
 		1000 0006  0P6S (6 port 16550a serial card)
+# Serial ports at BAR0-3, Parallel port at BAR4
+		1000 0014  1P4S (1 Parallel / 4 16550A Serial Port Adapter)
 	9855  PCI 9855 Multi-I/O Controller
+# Parallel port at BAR0. Serial ports at BAR2-5
 		1000 0014  1P4S
+# Parallel ports at BAR0,BAR2. Serial ports at BAR4-5
+		1000 0022  2P2S (2 Parallel / 2 16550A Serial Port Adapter)
 	9865  PCI 9865 Multi-I/O Controller
 	9901  PCIe 9901 Multi-I/O Controller
 9902  Stargen Inc.
@@ -16888,6 +17251,8 @@
 	a099  2000PV [Stingray]
 	a0a1  2000MT
 	a0a9  2000MI
+# Found on M2N68-AM Motherboard
+f043  ASUSTeK Computer Inc. (Wrong ID)
 f05b  Foxconn International, Inc. (Wrong ID)
 f1d0  AJA Video
 	c0fe  Xena HS/HD-R
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/pcilib.7 work/main.d/pciutils-3.1.2/pcilib.7
--- tmp/pciutils-3.1.2/pcilib.7	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/pcilib.7	2009-04-29 10:31:40.000000000 -0300
@@ -0,0 +1,110 @@
+.TH pcilib 7 "01 February 2009" "pciutils-3.1.2" "The PCI Utilities"
+.IX pcilib
+.SH NAME
+pcilib \- a library for accessing PCI devices
+
+.SH DESCRIPTION
+
+The PCI library (also known as \fIpcilib\fP and \fIlibpci\fP) is a portable library
+for accessing PCI devices and their configuration space.
+
+.SH ACCESS METHODS
+
+.PP
+The library supports a variety of methods to access the configuration space
+on different operating systems. By default, the first matching method in this
+list is used, but you can specify override the decision (see the \fB-A\fP switch
+of \fIlspci\fP).
+
+.TP
+.B linux-sysfs
+The
+.B /sys
+filesystem on Linux 2.6 and newer. The standard header of the config space is available
+to all users, the rest only to root. Supports extended configuration space, PCI domains,
+VPD (from Linux 2.6.26), physical slots (also since Linux 2.6.26) and information on attached
+kernel drivers.
+.TP
+.B linux-proc
+The
+.B /proc/bus/pci
+interface supported by Linux 2.1 and newer. The standard header of the config space is available
+to all users, the rest only to root.
+.TP
+.B intel-conf1
+Direct hardware access via Intel configuration mechanism 1. Available on i386 and compatibles
+on Linux, Solaris/x86, GNU Hurd and Windows. Requires root privileges.
+.TP
+.B intel-conf2
+Direct hardware access via Intel configuration mechanism 2. Available on i386 and compatibles
+on Linux, Solaris/x86, GNU Hurd and Windows. Requires root privileges. Warning: This method
+is able to address only the first 16 devices on any bus and it seems to be very
+unreliable in many cases.
+.TP
+.B fbsd-device
+The
+.B /dev/pci
+device on FreeBSD. Requires root privileges.
+.TP
+.B aix-device
+Access method used on AIX. Requires root privileges.
+.TP
+.B nbsd-libpci
+The
+.B /dev/pci0
+device on NetBSD accessed using the local libpci library.
+.TP
+.B obsd-device
+The
+.B /dev/pci
+device on OpenBSD. Requires root privileges.
+.TP
+.B dump
+Read the contents of configuration registers from a file specified in the
+.B dump.name
+parameter. The format corresponds to the output of \fIlspci\fP \fB-x\fP.
+
+.SH PARAMETERS
+
+.PP
+The library is controlled by several parameters. They should have sensible default
+values, but in case you want to do something unusual (or even something weird),
+you can override them (see the \fB-O\fP switch of \fIlspci\fP).
+
+.SS Parameters of specific access methods
+
+.TP
+.B dump.name
+Name of the bus dump file to read from.
+.TP
+.B fbsd.path
+Path to the FreeBSD PCI device.
+.TP
+.B nbsd.path
+Path to the NetBSD PCI device.
+.TP
+.B obsd.path
+Path to the OpenBSD PCI device.
+.TP
+.B proc.path
+Path to the procfs bus tree.
+.TP
+.B sysfs.path
+Path to the sysfs device tree.
+
+.SS Parameters for resolving of ID's via DNS
+.TP
+.B net.domain
+DNS domain containing the ID database.
+.TP
+.B net.cache_name
+Name of the file used for caching of resolved ID's.
+
+.SH SEE ALSO
+
+.BR lspci (8),
+.BR setpci (8),
+.BR update-pciids (8)
+
+.SH AUTHOR
+The PCI Utilities are maintained by Martin Mares <mj@ucw.cz>.
Binary files tmp/pciutils-3.1.2/setpci and work/main.d/pciutils-3.1.2/setpci differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/setpci.8 work/main.d/pciutils-3.1.2/setpci.8
--- tmp/pciutils-3.1.2/setpci.8	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/setpci.8	2009-04-29 10:31:40.000000000 -0300
@@ -0,0 +1,181 @@
+.TH setpci 8 "01 February 2009" "pciutils-3.1.2" "The PCI Utilities"
+.IX setpci
+.SH NAME
+setpci \- configure PCI devices
+.SH SYNOPSIS
+.B setpci
+.RB [ options ]
+.B devices
+.BR operations ...
+
+.SH DESCRIPTION
+.PP
+.B setpci
+is a utility for querying and configuring PCI devices.
+
+All numbers are entered in hexadecimal notation.
+
+Root privileges are necessary for almost all operations, excluding reads
+of the standard header of the configuration space on some operating systems.
+Please see
+.BR lspci(8)
+for details on access rights.
+
+.SH OPTIONS
+
+.SS General options
+.TP
+.B -v
+Tells
+.I setpci
+to be verbose and display detailed information about configuration space accesses.
+.TP
+.B -f
+Tells
+.I setpci
+not to complain when there's nothing to do (when no devices are selected).
+This option is intended for use in widely-distributed configuration scripts
+where it's uncertain whether the device in question is present in the machine
+or not.
+.TP
+.B -D
+`Demo mode' -- don't write anything to the configuration registers.
+It's useful to try
+.B setpci -vD
+to verify that your complex sequence of
+.B setpci
+operations does what you think it should do.
+.TP
+.B --version
+Show
+.I setpci
+version. This option should be used stand-alone.
+.TP
+.B --help
+Show detailed help on available options. This option should be used stand-alone.
+.TP
+.B --dumpregs
+Show a list of all known PCI registers and capabilities. This option should be
+used stand-alone.
+
+.SS PCI access options
+.PP
+The PCI utilities use the PCI library to talk to PCI devices (see
+\fBpcilib\fP(7) for details). You can use the following options to
+influence its behavior:
+.TP
+.B -A <method>
+The library supports a variety of methods to access the PCI hardware.
+By default, it uses the first access method available, but you can use
+this option to override this decision. See \fB-A help\fP for a list of
+available methods and their descriptions.
+.TP
+.B -O <param>=<value>
+The behavior of the library is controlled by several named parameters.
+This option allows to set the value of any of the parameters. Use \fB-O help\fP
+for a list of known parameters and their default values.
+.TP
+.B -H1
+Use direct hardware access via Intel configuration mechanism 1.
+(This is a shorthand for \fB-A intel-conf1\fP.)
+.TP
+.B -H2
+Use direct hardware access via Intel configuration mechanism 2.
+(This is a shorthand for \fB-A intel-conf2\fP.)
+.TP
+.B -G
+Increase debug level of the library.
+
+.SH DEVICE SELECTION
+.PP
+Before each sequence of operations you need to select which devices you wish that
+operation to affect.
+.TP
+.B -s [[[[<domain>]:]<bus>]:][<slot>][.[<func>]]
+Consider only devices in the specified domain (in case your machine has several host bridges,
+they can either share a common bus number space or each of them can address a PCI domain
+of its own; domains are numbered from 0 to ffff), bus (0 to ff), slot (0 to 1f) and function (0 to 7).
+Each component of the device address can be omitted or set to "*", both meaning "any value". All numbers are
+hexadecimal.  E.g., "0:" means all devices on bus 0, "0" means all functions of device 0
+on any bus, "0.3" selects third function of device 0 on all buses and ".4" matches only
+the fourth function of each device.
+.TP
+.B -d [<vendor>]:[<device>]
+Select devices with specified vendor and device ID. Both ID's are given in
+hexadecimal and may be omitted or given as "*", both meaning "any value".
+.PP
+When
+.B -s
+and
+.B -d
+are combined, only devices that match both criteria are selected. When multiple
+options of the same kind are specified, the rightmost one overrides the others.
+
+.SH OPERATIONS
+.PP
+There are two kinds of operations: reads and writes. To read a register, just specify
+its name. Writes have the form
+.IR name = value , value ...
+where each
+.I value
+is either a hexadecimal number or an expression of type
+.IR data : mask
+where both
+.I data
+and
+.I mask
+are hexdecimal numbers. In the latter case, only the bits corresponding to binary
+ones in the \fImask\fP are changed (technically, this is a read-modify-write operation).
+
+.PP
+There are several ways how to identity a register:
+.IP \(bu
+Tell its address in hexadecimal.
+.IP \(bu
+Spell its name. Setpci knows the names of all registers in the standard configuration
+headers. Use `\fBsetpci --dumpregs\fP' to get the complete list.
+See PCI bus specifications for the precise meaning of these registers or consult
+\fBheader.h\fP or \fB/usr/include/pci/pci.h\fP for a brief sketch.
+.IP \(bu
+If the register is a part of a PCI capability, you can specify the name of the
+capability to get the address of its first register. See the names starting with
+`CAP_' or `ECAP_' in the \fB--dumpregs\fP output.
+.IP \(bu
+If the name of the capability is not known to \fBsetpci\fP, you can refer to it
+by its number in the form CAP\fBid\fP or ECAP\fBid\fP, where \fBid\fP is the numeric
+identifier of the capability in hexadecimal.
+.IP \(bu
+Each of the previous formats can be followed by \fB+offset\fP to add an offset
+(a hex number) to the address. This feature can be useful for addressing of registers
+living within a capability, or to modify parts of standard registers.
+\IP \(bu
+Finally, you should append a width specifier \fB.B\fP, \fB.W\fP, or \fB.L\fP to choose
+how many bytes (1, 2, or 4) should be transferred. The width can be omitted if you are
+accessing a named register whose width is well known.
+
+.PP
+All names of registers and width specifiers are case-insensitive.
+
+.SH
+EXAMPLES
+
+.IP COMMAND
+asks for the word-sized command register.
+.IP 4.w
+is a numeric address of the same register.
+.IP COMMAND.l
+asks for a 32-bit word starting at the location of the command register,
+i.e., the command and status registers together.
+.IP VENDOR_ID+1.b
+specifies the upper byte of the vendor ID register (remember, PCI is little-endian).
+.IP CAP_PM+2.w
+corresponds to the second word of the power management capability.
+.IP ECAP108.l
+asks for the first 32-bit word of the extended capability with ID 0x108.
+
+.SH SEE ALSO
+.BR lspci (8),
+.BR pcilib (7)
+
+.SH AUTHOR
+The PCI Utilities are maintained by Martin Mares <mj@ucw.cz>.
Binary files tmp/pciutils-3.1.2/setpci.o and work/main.d/pciutils-3.1.2/setpci.o differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/update-pciids work/main.d/pciutils-3.1.2/update-pciids
--- tmp/pciutils-3.1.2/update-pciids	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/update-pciids	2009-04-29 10:31:40.000000000 -0300
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+[ "$1" = "-q" ] && quiet=true || quiet=false
+
+set -e
+SRC="http://pciids.sourceforge.net/v2.2/pci.ids"
+DEST=/usr/share/pci.ids
+PCI_COMPRESSED_IDS=
+GREP=grep
+
+# if pci.ids is read-only (because the filesystem is read-only),
+# then just skip this whole process.
+if ! touch ${DEST} >/dev/null 2>&1 ; then
+	${quiet} || echo "${DEST} is read-only, exiting." 1>&2
+	exit 1
+fi
+
+if [ -n "$PCI_COMPRESSED_IDS" ] ; then
+	DECOMP="cat"
+	SRC="$SRC.gz"
+	GREP=zgrep
+elif which bzip2 >/dev/null 2>&1 ; then
+	DECOMP="bzip2 -d"
+	SRC="$SRC.bz2"
+elif which gzip >/dev/null 2>&1 ; then
+	DECOMP="gzip -d"
+	SRC="$SRC.gz"
+else
+	DECOMP="cat"
+fi
+
+if which curl >/dev/null 2>&1 ; then
+	DL="curl -o $DEST.new $SRC"
+    ${quiet} && DL="$DL -s -S"
+elif which wget >/dev/null 2>&1 ; then
+	DL="wget --no-timestamping -O $DEST.new $SRC"
+	${quiet} && DL="$DL -q"
+elif which lynx >/dev/null 2>&1 ; then
+	DL="eval lynx -source $SRC >$DEST.new"
+else
+	echo >&2 "update-pciids: cannot find curl, wget or lynx"
+	exit 1
+fi
+
+if ! $DL ; then
+	echo >&2 "update-pciids: download failed"
+	rm -f $DEST.new
+	exit 1
+fi
+
+if ! $DECOMP <$DEST.new >$DEST.neww ; then
+	echo >&2 "update-pciids: decompression failed, probably truncated file"
+	exit 1
+fi
+
+if ! $GREP >/dev/null "^C " $DEST.neww ; then
+	echo >&2 "update-pciids: missing class info, probably truncated file"
+	exit 1
+fi
+
+if [ -f $DEST ] ; then
+	mv $DEST $DEST.old
+	# --reference is supported only by chmod from GNU file, so let's ignore any errors
+	chmod -f --reference=$DEST.old $DEST.neww 2>/dev/null || true
+fi
+mv $DEST.neww $DEST
+rm $DEST.new
+
+# Older versions did not compress the ids file, so let's make sure we
+# clean that up.
+if [ ${DEST%.gz} != ${DEST} ] ; then
+	rm -f ${DEST%.gz} ${DEST%.gz}.old
+fi
+
+${quiet} || echo "Done."
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/pciutils-3.1.2/update-pciids.8 work/main.d/pciutils-3.1.2/update-pciids.8
--- tmp/pciutils-3.1.2/update-pciids.8	1969-12-31 21:00:00.000000000 -0300
+++ work/main.d/pciutils-3.1.2/update-pciids.8	2009-04-29 10:31:40.000000000 -0300
@@ -0,0 +1,34 @@
+.TH update-pciids 8 "01 February 2009" "pciutils-3.1.2" "The PCI Utilities"
+.IX update-pciids
+
+.SH NAME
+update-pciids \- download new version of the PCI ID list
+
+.SH SYNOPSIS
+.B update-pciids
+.RB [ -q ]
+
+.SH DESCRIPTION
+.B update-pciids
+fetches the current version of the pci.ids file from the primary distribution
+site and installs it.
+
+This utility requires curl, wget or lynx to be installed. If gzip or bzip2
+are available, it automatically downloads the compressed version of the list.
+
+.SH OPTIONS
+.TP
+.B -q
+Be quiet and do not report anything except errors.
+
+.SH FILES
+.TP
+.B /usr/share/pci.ids
+Here we install the new list.
+
+.SH SEE ALSO
+.BR lspci (8),
+.BR setpci (8)
+
+.SH AUTHOR
+The PCI Utilities are maintained by Martin Mares <mj@ucw.cz>.
